<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 12">
<meta name=Originator content="Microsoft Word 12">
<link rel=File-List href="data_classification_qsar_files/filelist.xml">
<link rel=Edit-Time-Data href="http://biocomp.health.unm.edu/biomed505/Course/Cheminformatics/advanced/data_classification_qsar/data_classification_qsar_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>every1</o:Author>
  <o:Template>Normal</o:Template>
  <o:LastAuthor>Dan Fara</o:LastAuthor>
  <o:Revision>2</o:Revision>
  <o:TotalTime>987</o:TotalTime>
  <o:LastPrinted>2008-01-17T10:52:00Z</o:LastPrinted>
  <o:Created>2008-01-17T10:53:00Z</o:Created>
  <o:LastSaved>2008-01-17T10:53:00Z</o:LastSaved>
  <o:Pages>19</o:Pages>
  <o:Words>6900</o:Words>
  <o:Characters>39331</o:Characters>
  <o:Lines>327</o:Lines>
  <o:Paragraphs>92</o:Paragraphs>
  <o:CharactersWithSpaces>46139</o:CharactersWithSpaces>
  <o:Version>12.00</o:Version>
 </o:DocumentProperties>
</xml><![endif]-->
<link rel=dataStoreItem href="data_classification_qsar_files/item0001.xml"
target="data_classification_qsar_files/props0002.xml">
<link rel=themeData href="http://biocomp.health.unm.edu/biomed505/Course/Cheminformatics/advanced/data_classification_qsar/data_classification_qsar_files/themedata.thmx">
<link rel=colorSchemeMapping
href="data_classification_qsar_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:DrawingGridHorizontalSpacing>5.5 pt</w:DrawingGridHorizontalSpacing>
  <w:DisplayHorizontalDrawingGridEvery>2</w:DisplayHorizontalDrawingGridEvery>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:DontUseIndentAsNumberingTabStop/>
   <w:FELineBreak11/>
   <w:WW11IndentRules/>
   <w:DontAutofitConstrainedTables/>
   <w:AutofitLikeWW11/>
   <w:HangulWidthLikeWW11/>
   <w:UseNormalStyleForList/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="--"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="true"
  DefSemiHidden="true" DefQFormat="false" DefPriority="99"
  LatentStyleCount="267">
  <w:LsdException Locked="false" Priority="0" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" Priority="0" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" Name="toc 9"/>
  <w:LsdException Locked="false" Priority="35" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" Priority="10" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" Priority="11" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" Priority="0" Name="Body Text Indent 2"/>
  <w:LsdException Locked="false" Priority="22" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="59" SemiHidden="false"
   UnhideWhenUsed="false" Name="Table Grid"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" UnhideWhenUsed="false" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" SemiHidden="false"
   UnhideWhenUsed="false" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" SemiHidden="false"
   UnhideWhenUsed="false" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" SemiHidden="false"
   UnhideWhenUsed="false" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" SemiHidden="false"
   UnhideWhenUsed="false" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" SemiHidden="false"
   UnhideWhenUsed="false" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;
	mso-font-charset:2;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:0 268435456 0 0 -2147483648 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-1610611985 1107304683 0 0 159 0;}
@font-face
	{font-family:"Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:128;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-134238209 -371195905 63 0 4129279 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-1610611985 1073750139 0 0 159 0;}
@font-face
	{font-family:"\@Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:128;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-134238209 -371195905 63 0 4129279 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:Calibri;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{mso-style-update:auto;
	mso-style-noshow:yes;
	mso-style-unhide:no;
	mso-style-next:Normal;
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Footnote Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:Calibri;}
span.MsoFootnoteReference
	{mso-style-noshow:yes;
	mso-style-priority:99;
	vertical-align:super;}
span.MsoEndnoteReference
	{mso-style-noshow:yes;
	mso-style-priority:99;
	vertical-align:super;}
p.MsoEndnoteText, li.MsoEndnoteText, div.MsoEndnoteText
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Endnote Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:Calibri;}
p.MsoBodyTextIndent2, li.MsoBodyTextIndent2, div.MsoBodyTextIndent2
	{mso-style-noshow:yes;
	mso-style-unhide:no;
	mso-style-link:"Body Text Indent 2 Char";
	margin:0in;
	margin-bottom:.0001pt;
	text-align:justify;
	text-indent:.5in;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:purple;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p
	{mso-style-priority:99;
	mso-margin-top-alt:auto;
	margin-right:0in;
	mso-margin-bottom-alt:auto;
	margin-left:0in;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
span.EndnoteTextChar
	{mso-style-name:"Endnote Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Endnote Text";}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Footnote Text";}
span.BodyTextIndent2Char
	{mso-style-name:"Body Text Indent 2 Char";
	mso-style-noshow:yes;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Body Text Indent 2";
	mso-ansi-font-size:11.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	mso-fareast-font-family:Calibri;}
 /* Page Definitions */
 @page
	{mso-footnote-separator:url("data_classification_qsar_files/header.htm") fs;
	mso-footnote-continuation-separator:url("data_classification_qsar_files/header.htm") fcs;
	mso-endnote-separator:url("data_classification_qsar_files/header.htm") es;
	mso-endnote-continuation-separator:url("data_classification_qsar_files/header.htm") ecs;
	mso-endnote-numbering-style:arabic;}
@page Section1
	{size:8.5in 11.0in;
	margin:.5in .5in .5in .5in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-paper-source:0;}
div.Section1
	{page:Section1;
	mso-endnote-numbering-style:arabic;}
 /* List Definitions */
 @list l0
	{mso-list-id:74203371;
	mso-list-type:hybrid;
	mso-list-template-ids:1459769530 67698689 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l1
	{mso-list-id:348459307;
	mso-list-type:hybrid;
	mso-list-template-ids:-1292975948 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l2
	{mso-list-id:555512304;
	mso-list-type:hybrid;
	mso-list-template-ids:-1003334538 67698703 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l2:level1
	{mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3
	{mso-list-id:899747083;
	mso-list-type:hybrid;
	mso-list-template-ids:630229904 67698703 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l3:level1
	{mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4
	{mso-list-id:944657900;
	mso-list-type:hybrid;
	mso-list-template-ids:605949050 67698689 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l5
	{mso-list-id:1188788145;
	mso-list-type:hybrid;
	mso-list-template-ids:629990760 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l5:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l6
	{mso-list-id:1197886338;
	mso-list-type:hybrid;
	mso-list-template-ids:-274162326 67698715 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l6:level1
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:.5in;
	mso-level-number-position:right;
	text-indent:-.25in;}
@list l7
	{mso-list-id:1607881650;
	mso-list-template-ids:-2014121618;}
@list l7:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:Symbol;}
@list l7:level2
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;
	mso-ansi-font-size:10.0pt;
	font-family:"Courier New";
	mso-bidi-font-family:"Times New Roman";}
@list l8
	{mso-list-id:1710178801;
	mso-list-type:hybrid;
	mso-list-template-ids:1679559714 67698703 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l8:level1
	{mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l9
	{mso-list-id:1952080027;
	mso-list-type:hybrid;
	mso-list-template-ids:-1265204080 67698703 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l9:level1
	{mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l10
	{mso-list-id:2004970411;
	mso-list-type:hybrid;
	mso-list-template-ids:-941201304 -1155752650 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l10:level1
	{mso-level-number-format:bullet;
	mso-level-text:-;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-qformat:yes;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";}
table.MsoTableGrid
	{mso-style-name:"Table Grid";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-priority:59;
	mso-style-unhide:no;
	border:solid black 1.0pt;
	mso-border-alt:solid black .5pt;
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-border-insideh:.5pt solid black;
	mso-border-insidev:.5pt solid black;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="2050"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link=blue vlink=purple style='tab-interval:.5in'>

<div class=Section1>

<p class=MsoNormal style='text-align:justify'><b style='mso-bidi-font-weight:
normal'><i style='mso-bidi-font-style:normal'><span style='font-size:16.0pt;
line-height:115%'>Section III</span></i></b><b style='mso-bidi-font-weight:
normal'><span style='font-size:16.0pt;line-height:115%'><span style='mso-tab-count:
1'> </span><o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:14.0pt;
line-height:115%'><o:p>&nbsp;</o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:14.0pt;
line-height:115%'>Cheminformatics - Advanced: </span></b><i style='mso-bidi-font-style:
normal'><span style='font-size:14.0pt;line-height:115%'>Data Analysis,
Classification Methods, Quantitative Structure-Activity Relationships (QSAR)</span></i><b
style='mso-bidi-font-weight:normal'><span style='font-size:14.0pt;line-height:
115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'><o:p></o:p></span></b></p>

<p class=MsoNormal style='text-align:justify'><i style='mso-bidi-font-style:
normal'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></i></p>

<p class=MsoNormal style='text-align:justify'><i style='mso-bidi-font-style:
normal'><span style='font-size:12.0pt;line-height:115%'>Dan C. Fara and Tudor
I. Oprea<o:p></o:p></span></i></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1. Methods for data analysis and classification</span></b><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'><o:p></o:p></span></p>

<p style='text-align:justify;line-height:115%'><span style='mso-bidi-font-weight:
bold'>Generally, <i style='mso-bidi-font-style:normal'>data analysis</i></span>
is the process of looking at and summarizing <span style='mso-bidi-font-weight:
bold'>data</span> with the intent to extract useful information and develop conclusions.
Data analysis is closely related to <i style='mso-bidi-font-style:normal'>data
mining</i>, but data mining tends to focus on larger data sets, with less
emphasis on making inference, and often uses data that was originally collected
for a different purpose. Some people divide data analysis into exploratory data
analysis (EDA) and confirmatory data analysis (CDA), where the EDA focuses on
discovering new features in the data, and CDA on confirming or falsifying
existing hypotheses.</p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Analysis of the data and, in
principal, establishment of a relationship between chemical compounds and their
activity / properties can be done by (i) a <i>statistical or
pattern-recognition</i> method, and (ii) a <i>learning</i> method.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.1 Statistical and/or pattern recognition methods<o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.1.1 Multivariate data </span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><i><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman"'>Multivariate data</span></i><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'> refer to objects, features and properties. An <i>object</i> is any
real or abstract item, in our case a chemical structure, which is characterized
by a set of features. A <i>feature</i> is a numerical variable (measured or
computed data) such as molecular descriptor. Application of statistical methods
requires a reasonable number of objects and features, described by an (<i>n</i>
x <i>m</i>) matrix X, containing a row for each of the <i>n</i> objects, and a
column for each of the <i>m</i> features. In a geometrical interpretation each
object is considered as a point (or a vector), with the features used as
coordinates (or vector components) in an <i>m</i>-dimensional <i>feature space</i>.
The applied hypothesis assumes that objects close together have attributes in
common (for instance a similar value for the biological activity or
physicochemical property or the same substance class). Additionally, a <i>y</i>-column
or a Y-matrix might contain data for one or several properties of the objects.
A <i>property</i> is an interesting attribute of the objects difficult or not
directly measurable, i.e. biological activity. The goal of multivariate
analysis is to model and predict the property <i>y</i> from the features <i>x</i>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>In other words, the aim of
multivariate data analysis is to find a formal combination (a mathematical
function or a more general algorithm) of the features by defining a new,
so-called <i>latent variable </i>(<i>score</i>) which reveals problem-relevant
information. The coefficients of the features are called <i>loadings</i> and
the values of the latent variable are called <i>scores</i>. The loadings
describe the individual contributions of the feature to the latent variables. Depending
on the aim of data analysis different mathematical criteria are applied:<o:p></o:p></span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l4 level1 lfo8'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>For
     principal components the criterion is maximum variance of the score,
     providing an optimum representation of the Euclidean distances between the
     objects;<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l4 level1 lfo8'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>For
     a discriminant variable an optimum separation of object classes is
     required;<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l4 level1 lfo8'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>Latent
     variables with maximum correlation to a property of interest, <b
     style='mso-bidi-font-weight:normal'>y</b>, are used in multivariate
     calibration.<o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Principal Component Analysis
(PCA)</span></u><a name="_Ref188233245"></a><a style='mso-endnote-id:edn1'
href="data_classification_qsar.htm#_edn1" name="_ednref1" title=""><span style='mso-bookmark:_Ref188233245'><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[1]</span></span><![endif]></span></span></span></span></a><span
style='mso-bookmark:_Ref188233245'></span><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'> is the most frequently used method for computing linear latent
variables. Latent variables from PCA optimally represent the distances between
objects in a multivariate data set X. The direction that best describes the
relative distances between the objects is the latent variable which has <i>maximum
variance</i>. This direction is called the <i>first principal component</i>
(PC1). The second principal component (PC2) is <b style='mso-bidi-font-weight:
normal'>orthogonal</b> to PC1 and again has the maximum possible variance.
Further principal components can be determined by continuing this concept.
Determination of all principal components corresponds to a rotation of the
coordinate system of the feature space – see Figure 1.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Object_x0020_1" o:spid="_x0000_i1026" type="#_x0000_t75"
 style='width:413.25pt;height:408pt;visibility:visible' o:gfxdata="UEsDBBQABgAIAAAAIQCTXr2S+wAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWxQ5dIISSdEHKEhAqBxjZk8RqMrY8JrS3x2nLggVFLC3r/f++plrvp1HMGNl5quWtKqVA
Mt466mv5vn0q7qXgBGRh9IS1PCDLdXN9VW0PAVlkmriWQ0rhQWs2A07Aygek/NP5OEHKz9jrAGYH
PepVWd5p4ykhpSItGbKpXrJAdBbFK8T0DFPu0TayxpVvvVE5SorHE7PU1hJCGJ2BlKX1TFZNXPiu
cwZVG3lzpG4WSjdVix18jEls9rnytDLiyH8E/lxQnO1VJo+lPLjAFxouG5/Nfltt/SdFnP87u83Y
G87fVvp4oOYLAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJlbHOk
kMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qRJVI2
sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY22Yi
Tra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1iOWA1
4Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAAIQBE
lgsR2hIAAHKLAQAOAAAAZHJzL2Uyb0RvYy54bWzsnV9z2kYXxu/fmfc7aHTbSY3+gyek88Zt815k
Wk+c9n4NwjAREiMphPTT95zd1UoQYAET27RPLhzZSIt4ONo9vz1nz77+aTXPnGVaVrMiH7rejz3X
SfNRMZ7lD0P3j4+/vuq7TlWLfCyyIk+H7te0cn9689//vP6yuE79Ylpk47R0qJG8uv6yGLrTul5c
X11Vo2k6F9WPxSLN6cVJUc5FTb+WD1fjUnyh1ufZld/rxVdfinK8KItRWlX015/Vi+4b2f5kko7q
3yeTKq2dbOjSvdXyZyl/3vPPqzevxfVDKRbT2UjfhjjhLuZiltObmqZ+FrVwPpezE5rKitGndHwj
8qWoqMlsdN39i77HbPT4lsV1vnxXLu4WtyXf+ei35W3pzMZSm1zM6ctS6tAL+jQS62rjqoe2gdWk
nHNDxWTirGQrX43C6ap2RvTHyI96QT9wnRG9Fnl9L+7p72A0pS+Kr/PCMOnTXx0+IwjC9oRfLI3Q
val7oIPOfclDvrEdH7efuI76vO/K4vPC6ceP+ty77v8lS0BfSFeB0HucAknk8/fK32A/6A8GoWqu
USAMvYF8nY0g7Cd03EhujGBHE6NpYwS7GjnRCIJ4Q4LGLE8zfm/H/TcS+L7Xi0gX+RyEPerJouYN
D5dgVyM7JBit8ruFeg5u+HDzsfe8RoK7uhSzh2nt3BR5Tl1oUTpec3u/LfXFTWfQaavSbfJT6JQF
dbYRPb70T3aNul/wk17kaevwe4Moah5wo03khcY8vKjfZ+sxn0lcL8qqfpcWc4cPhm42y1PZvli+
r2p1anMKf9os55/TVIx/ycdO/XVBHVtdzkT+kKXKLmsxy9rXchqn9DvypfTGzceq6q9Zqpr8kE6o
q+TeSr61HK7Sm6x0loIGGjEapXktHyK6PMvpbL5sMssyc6HSZO+F+ny+NJVD2TEXmyvkOxd5bS6e
z/Ki3Hbb9aq55Yk6X3169anpgxgDsvWoMRl2tz8JBkroE0eS7sPk+QO2KdWesRiyIzlkcIcSRHEQ
h7rLaUeVzgO51kbbo/g7WjGmRwedYcWokTdPBCvdjqKmQ9nyNEVGj0c+TJ4fJYF+mIIwTvjBokeA
DEaPlX4vSlgv2dE89mEiAfAsUA/QfRZsZkD+r3oUtphBcqwZTLLZ4s+m09H9affpCIIg6jW2b0yg
a9de5HOXzG9s7BpdqnRX+bHZ2x9/zy7VZkaev8eOaOSWDz11Pgf2J9sMyff7NOwqt22ti2wMyYuT
uO1LvJA6nkcZErqTzaGVFNFja3vUGXd3jDS+AZgtfYzfuNaH2gY/BrprCQYx9xbSkT/AInpxzKPw
I7oWWMSmRRzw7Q/29Ay+dJ6lW3Jgz7DLbQ+jhChF2UIQ+gP97Hc8DS8Izua2wxCON4RgH78F2i88
eIjY0Q3QhEww8H013jQDw5rnfEYPYx3MutBG9gE444fkBDPZ50mQDR3nSXTMpOs/hOEg0mNB20Os
+w++N6AOBaPFedHc5kcGBs23+ApBcOS372zzI7teA+yA4hi7Joa+H09Qz2DmaCo95/fthF9opn1/
p2krJ9S9Og0RfK5y43J5LNGoM7nXfeoHiddM12//tiVxNhMQ0p/c6yGmGUFutXdKLy9+pak01Uoz
IVGv1ORmvXpbjL/yDd7T/xTRKOvsplDRH5GPpgUFf0Z1qQw9q+o71YOK63E6uVVTonT0gS7MaIpw
6Kb5qz/udC/VnpItM4/0cOaifC9DHSJ7oPhX5jp0zkdxf/fX0B1QJIAdJroBeUoq3udvy0+M79R2
Pcv1r3TKlN6Kole3n/MRzWiqOUV9FxW1pAT+lJYcYuNOU34fRTYbsw7yF46XtXaWmcnHqnuWfFc5
BToRI5oG/WGev8pqJUUqNl5IhXphVG28MKpaOUgnOZGg9eBboUO/lUa5ja4DfVgUrU/Q6tMYCfRh
UbQ+YauPFyQyRggDylgVLVDUEajv92X/CwtiVbRAcSuQcUshEKuiBUo6AiVhwIMKHrGMVdEC9VuB
WB05ksOCWBUt0KAjUBxxEB8WtMxYFeURdVxL6YGzXMqAjPuZ5uNbUYrtziZdRG58685+rtK7xQeK
xiu/t/F3twWlfZe/ox2xDPnSVIxTFauOaCKvoXBzhfoEW+PWBwS8nzpu3Xq72+PWioD2cJBhYsVB
LQUfwUFBvx/LXpRD0PqYvqnO7Me5OYinf/Tj2BhDay2AH5YH8LOlI2gfl6fTB/CjcXsHPAN+mvmI
HQIBfiwCAX4sAgF+LAIBfiwCAX4sAgF+TJKv9PyfImm39eZOhJ+oSR9R8NMkzh4VBDIzJDqh/9vQ
77nhh1ZzIAi0QTgG3xXZt5bxdH4+B8XY9BEE2h4kAweBg2gFSht+PTbKCg6y+CDgIItA4CCLQOAg
i0DgIItA4KAL5CCzXk9xULtE75ggUBMpZg6ihe4cFP2+QSBwkIp1diM94CDXeeHJguAgcBA4iJOA
N5Jrz5aOCw6yOGngIItA4CCLQOAgi0DgoAvkILOYXHFQu3z8CA5SdbsoIZQ4yI/j5JvFwmqRKBYF
OVgU9K9eNAUOAgeBg8BBunDcc4TswUEWLxYcZBEIHGQRCBx0gRxkCvcpDmpr9R3BQd1SGFSZL+IF
QogHoTiCrhuB4ghN8QhwEDgIHAQOAge93AI/4CCLmw8OsggEDrpADjLlZRUHtQVlj+Ag2vCDKtOr
eFDg+SE4CEXi2OUHB3HZwU4RPXAQOAgcBA4CB4GDLrYSLDgIHLSl7tT6rmiXViSO9ufTGzkpDmq3
NDuGg6gwgiyUyEXitm2igbw4FMtGGU1VQVzXkQhaTkSx7CZexqJofVAsWxVEXQdp1Emw+CBYH2QR
CHlxFoEQD7IIBA6yCIR40OXFgyKzy5zkILNt+VH14sI47DfFsn29gRDy4pAXh3gQ4kGyHzh00ylw
kGWIBQdZBAIHWQQCB1kEAgdZBAIHWQQCB10gB5ltlBUHtRsnHxMPot205c5iqBeHzVNRN5u2o92x
/Qvy4pAXh7w45MUhLw55cciL+6duIQ8OukAOCtby4iJfFTgYHRcPihIKB+n1QX1eKoQ6CeIe8SDE
gxAPQjyIsPhsZaERD7LMxSIeZBEI8SCLQIgHWQRCPMgiEDjoAjkoXOeg4CQOGkSB4SDUSXAop/8j
OAh1Eq4EOAgcBA66frqdnMFBFicNHGQRCBxkEQgcZBEIHHSBHBStc1B4Egd16iT40aBHv6JuNuJB
qBcHDtLdSbWxb+aOvEGsD7IMsYgHWQQCB1kEAgdZBAIHWQQCB1kEAgddIAfF6xwUncRB3brZQeSD
gxAPQt1sWQRtvdwX1gdhfRDWB2F9ENYHYX0Q1gdhfdBrcZ3m41tRCtpcw8lE/jB00/zVH3fshVM2
yUL+rFdvi/FXXnf9uUrvFh/SUa1erld3dIa4ruqvWSrXZecf0okzG6+Gru/KV0bTdJ7eZLrDaWpb
y5emYpw6S5EN3ahZyUJvWTVXqBqu3CCfretgq7a9fW2rW7u4utnJOgfFp3BQUyvb4ToJeq2QzIdJ
V7Uzoi8FdbNRNxt1s1E3W1hm0hAPsgiEeJBFIMSDLAIhHmQRCPEgi0CIB1kEQjzIcBMjVDqZELkZ
QOvZIcpcIQmsyNuL57O8KLc1kNWywhuB3ESdLzFOAyLjnQJG+TNfcg04bpvKIBB/zsaEgv11DkpO
4iBdG0FyUDiIEA9CPAjxIPkkIh4k+xvkxal5IbGRH5gK3d8eKBA4yOKDgIMsAoGDLAKBgywCgYMs
AoGDLpCDBusc1D+Jg6Je1OwfFMRRwnuqIh6EenGoF4c6CbIfwP5Buj8EB4nrDHlxyItDXhzy4pAX
x9ES5MVRZl6TsEfeAgVvniEeFFO161zMKW1f7R80OImDNPtwPCgYxD7vJQQOAgeBg8BB4CDUi0O9
uFmWcVzUJN+rvPw2mePpCuohHmSZzUc8yCIQ4kEWgRAPurx4UOytcRBhkXJcjto/qGEfmRfneSHy
4pAXh7w45MW9OjbtC+uDLEMs8uIsAiEvziIQOMgiEDjIIhA4yCLQURwk16mUG1USnPGsrCmoQEGK
al7fZKnI+RfpmtdvlksOmtTSv5CX74gpta3QhXQFqi6Y1UOq3sN3W20U++tUJdcuyeVIvDJJfRvt
KqVKr1ZaTco5z5YVk4lDRRWCqO9xRElSlR/zFq2ILqEKN6pwowq3nqQ5cDENqMoyYIOqLAKBqiwC
gaosAoGqLAKBqiwCHUVVO4gIHMSlFpgxdBWF781BwToH+dpxOSq6FGr2kVl2SRAguoToEqJLiC4h
uqSLjme1nhfCaiOsNmqL4iLLjjMPi2w2/vXlpCGCgyxuPjjIIhA46KVm2Y1Wuapbni9v+HCzBl1s
9ma9q0sxe5jWzk2R51Q8ryiduN2pVV/chIk6bXUjRU5Z0JLCiCJE9M91Jtls8X+5wo6P/uSjbizJ
88OmYoOXRP43FRuCHmftUTsjXsukf1F30ISlFmVVv0uLucMHQzeb5al8C7F8X+nC7c0p/M5Zzj/5
Lz+LaqoSv8d0xL4KBSP5Zfqv+UTbSr0fUI4921rH/YAShE9dx71e7a9faGzHHOwwooS+I7VwbZsR
tcvYjjEiLybbMFa0aTveoO8Z29HH9B3Ssj1d/d1PwkCewLbT/ALb2dycQClywlrH89lOuMd2ksBM
zhxoO1s7mkG/50W0+Rp3I6E/YKtaM5aIXg3pPthWIq8XxrICDfUEJ3czzheyOu4HZW9Ui1n2Sz6W
9eeGbl3OaPOJLEWnI4usqi5X7apBkpu+xhzs6nTMbnpbOp2k3VvvQMORI9f+TqexHW1HQd/X42PT
6URBIJMjpB2p40d2ObCjXcV3D+6AVBXeNrOlrb+bmLpTH+kbfFusnGS99JTDu8HwViL6UW0baVyE
poPQ6TEe+TDNsMS+y7cuTSzdHOXRxIMgkLPPu3uakhyxfQ5NXjBHKiNr7qnZqabdyuaetrShbK4v
pVgM3bwgH8kp6+ymoEXv2i1Z/O9zTU1pl0mdLx2mqr5rNryhcqq3yn1UhVW37KHTnpItM4+cTb3q
mPwDkT1QvljmOnTOR/HcKTOt9azB+HctC8N6SE3VnhwvekH2c+hzSRs2Poc+QWs/nByi7ccLEu51
8IAts4tKmXkOC7qolJnnEOiiUmaeQ6CLmip+DoEuaqr4OQTaPVUsnaPjVh2sNhYdbG/iXlQpzwwO
3Vc+b79IPui+pQzeRqM78nacPc0qh/ygtQ1yvpHdwt2c0jfTa4ZT2jm1O3YFjuWUoN/n6TU5I6Kh
dW1CJACmAFOAKRr7Fe6qCMFl7af4HB08MMUSLAWmWAQCplgEAqZYBAKmWAQCplgEelmYIp3z+k2w
RiZEDmcjDFNxqSEMYg75phQtOYkw4h6F6DVhIBLSTtdiolZHhhAJoVqkowqIQTL8o0syPQeDATEs
wzsQwyIQEMMiEBDDIhAQwyLQi0QM/7shhik/ZBBjvQLR0UGM2ItC+ieDGF6UBFEcK2Yx6XhhkASU
4yXTx5FsFcrcREr8kjW8UvE+f1t+eor9b57DA0KylV6ICcRwfwBiiGYs+neuy32OHgiI0djcji4I
iGERCIhhEQiIYRHorIhxe3NoZpRJrzpzHpSp3GMQYr14z9EI4YVh0kQpti5BjYAQSIRCIhSiFEAI
IIR+Cp5uxRgQwuLfACEsAgEhLAIBISwCnRkh1uMLO7ceb1donBkhTLUJgxBtiYmTEp2CXtIb0DJu
WRQgiHzGCZk51UQhQj/yPKpVgCjEX0N3QMTFWWGIQiARjBLBsOR7f5QGayksoxMSnSwCASEsAgEh
LAIBISwCASEsAh2LEPcyKcUEEbZtN7c4lCM22jozTZjimYYm2oqZJ9FEGHoDWc2FS9WBJkyVG3jL
WDYxMTPQOxIKQBNnpIm/BQAAAP//7JXPTsMwDMZfJcoVoa1l/0V7YIgTh2mFB/BS01Xz0ioJY+Pp
sdNKG7vwAOzSOHXyxfo19bcpvjM9T0aj4VArFyjTPCK82ie3y3SiFUGobT/l1BZsVdtq9WlNkHz+
CIsSP9YrpzwrJTPR2aGzHKccS943VJcvNVGcmC3ucUlOHYBPC8dED/LHwdWqeKoKpxY/wGCm7/b2
noKshAXCVQKhSxh/lTC+1+4qjMfQgUbtyokQh2MO1R7cq5SezmL1QBVXT1rxrjf4j4CESg9ocgaU
prPJUD7vDZBQ6QFNLwBNRw9y52+ASKj0gGZnQEInNprbDRIqPaD5BaDJeMr/2O0GHUiodA3bhyKc
CIVWK4/IzXHfJvaiTKO9fy+02ogbqbJ24RQ9zO/DkhC4kw87dwh5KzYTominEdX+EIqWE/Lk117W
4VJE7fjUlCfR+fRYtGs0oXOzcCy6Fb4bKtfGF2QW1JgdlkuwB/BRpHLQbmvzDAEu5xx/tQtMm21D
Jbr8BwAA//8DAFBLAwQUAAYACAAAACEAgwVuEtkAAAAFAQAADwAAAGRycy9kb3ducmV2LnhtbEyO
wU7DMAyG70i8Q+RJXBBLO4lqlKYTIO3AjW08QNa4TbXGSZtsK2+P4QIXy9b/6/NXbWY3iAtOsfek
IF9mIJAab3rqFHwetg9rEDFpMnrwhAq+MMKmvr2pdGn8lXZ42adOMIRiqRXYlEIpZWwsOh2XPiBx
1vrJ6cTn1Ekz6SvD3SBXWVZIp3viD1YHfLPYnPZnpyALfZGHbWvH9vD6dD9+6PedHZW6W8wvzyAS
zumvDD/6rA41Ox39mUwUAzO49zs5W6+KRxBHXvIiA1lX8r99/Q0AAP//AwBQSwECLQAUAAYACAAA
ACEAk169kvsAAADhAQAAEwAAAAAAAAAAAAAAAAAAAAAAW0NvbnRlbnRfVHlwZXNdLnhtbFBLAQIt
ABQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAAAAAAAAAAAAAACwBAABfcmVscy8ucmVsc1BLAQIt
ABQABgAIAAAAIQBElgsR2hIAAHKLAQAOAAAAAAAAAAAAAAAAACsCAABkcnMvZTJvRG9jLnhtbFBL
AQItABQABgAIAAAAIQCDBW4S2QAAAAUBAAAPAAAAAAAAAAAAAAAAADEVAABkcnMvZG93bnJldi54
bWxQSwUGAAAAAAQABADzAAAANxYAAAAA
">
 <v:imagedata src="data_classification_qsar_files/image001.png" o:title=""
  croptop="-2317f" cropbottom="-159f" cropleft="-1851f" cropright="-1876f"/>
 <o:lock v:ext="edit" aspectratio="f"/>
</v:shape><![endif]--><![if !vml]><img width=551 height=544
src="data_classification_qsar_files/image002.gif" v:shapes="Object_x0020_1"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>Figure 1.</span></b><span style='font-size:12.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>
Principal component analysis (PCA). The first principal component, PC1, is the
latent variable with maximum variance (loading vector <b style='mso-bidi-font-weight:
normal'>p<sub>1</sub></b>). PC2 is <b style='mso-bidi-font-weight:normal'>orthogonal</b>
to PC1 and again has maximum possible variance (loading vector <b
style='mso-bidi-font-weight:normal'>p<sub>2</sub></b>).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Partial Least Squares (PLS)
Regression</span></u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'> is a method of relating two data
matrices, X and Y<sup><span style='mso-field-code:" NOTEREF _Ref188233245 \\h &#1; \\* MERGEFORMAT "'>1<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200330033003200340035000000</w:data>
</xml><![endif]--></span></sup>. The criterion mostly applied in PLS is maximum
covariance between latent variables and <i>y</i>. This criterion combines high
variance of X (responsible for robustness) and high correlation with the
property of interest. PLS is a linear method (although non-linear versions
exist) and therefore the final latent variable that is used to predict property
<i>y</i>, is a linear combination of the features. X is a mean centered matrix
with <i>n</i> rows (objects) and <i>m</i> features (predictor variables); Y is
a mean centered matrix with <i>n</i> rows and <i>k</i> properties (response
variables). The PLS algorithm produces – similar to PCA but with a different
aim and with different methods – a score matrix T that models X and a score
matrix U that models Y; both have <i>n </i>rows and <i>a </i>columns with <i>a </i>being
the number of calculated PLS components (intermediate latent variables that are
regressed with <i>y</i>). T and U model X and Y, respectively, but also include
information about linear relationships between <i>x</i>- and <i>y</i>-data. The
<i>x</i>-scores are alinear combination of the <i>x</i>-variables and can be
regarded as good summaries of X. The -variables and can be regarded as good
summaries of X. The <i>y</i>-scores are a linear combination of the <i>y</i>-variables
and can be regarded as good summaries of Y. <i>x</i>- and <i>y</i>-scores are
connected by the inner (linear) relationships as described in reference<sup><span
style='mso-field-code:" NOTEREF _Ref188233245 \\h &#1; \\* MERGEFORMAT "'>1<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200330033003200340035000000</w:data>
</xml><![endif]--></span></sup>. Characteristic to the PLS algorithm is the
stepwise calculation of the components. After a component is computed the
residual matrices for X and Y are determined. The next PLS component is
calculated from the residual matrices and therefore its parameters (scores,
loadings, weights) do not relate to X but to the residual matrices. When a PLS
model has been created, it can be used to predict <i>y</i>-properties and <i>y</i>-scores
from new <i>x</i>-data. The model is a linear combination of the predictors.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.1.2 Multivariate exploratory data analysis<o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>If multivariate data only consist
of a feature matrix X, data analysis is often called <i>exploratory</i> or <i>unsupervised</i>,
because no additional information about the objects, i.e. compounds, is used or
available to guide the work. The purposes of data interpretation are manifold:
(i) obtaining a better insight into the data structure, (ii) searching for
groups with similar objects (<i>cluster analysis</i>), (iii) searching for
outliers, or (iv) searching for relevant features, i.e. descriptors.
Exploratory data analysis is usually the first step and represents an objective
inspection of data even if the final goal is <i>calibration</i> or <i>classification</i><a
style='mso-endnote-id:edn2' href="data_classification_qsar.htm#_edn2" name="_ednref2" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[2]</span></span><![endif]></span></span></a>. The
most used techniques are principal component analysis (PCA), cluster analysis
with dendrograms, i.e. agglomerative hierarchical cluster, and <i>Kohonen
mapping</i> (a non-linear method)<a name="_Ref188270291"></a><a
style='mso-endnote-id:edn3' href="data_classification_qsar.htm#_edn3" name="_ednref3" title=""><span
style='mso-bookmark:_Ref188270291'><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[3]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188270291'></span><sup>,<a style='mso-endnote-id:edn4'
href="data_classification_qsar.htm#_edn4" name="_ednref4" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[4]</span></span><![endif]></span></span></a></sup>.
<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>PCA</span></u><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'> Is usually the first method applied to a data
matrix X. Most powerful are scatter plots with a point for each object and the
coordinates of the points given by the scores of the first and second principal
components. This visualization of the data corresponds to a linear projection
of the <i>m</i>-dimensional features space on to a plane by preserving a
maximum of the relative distances (similarities) between the objects, i.e.
chemical compounds.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Hierarchical cluster analysis</span></u><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'> The starting point is a distance matrix containing
the distances between all possible pairs of objects. The pair with the smallest
distance is merged, and thereby the number of objects is reduced by one. The
distance matrix is updated and the procedure continued until all objects are
merged into a single cluster – example given in Figure 2<sup><span
style='mso-field-code:" NOTEREF _Ref188235746 \\h &#1; \\* MERGEFORMAT "'>5<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200330035003700340036000000</w:data>
</xml><![endif]--></span></sup>. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><!--[if gte vml 1]><v:shape id="_x0000_i1025"
 type="#_x0000_t75" style='width:449.25pt;height:480.75pt'>
 <v:imagedata src="data_classification_qsar_files/image003.emz" o:title=""/>
</v:shape><![endif]--><![if !vml]><img width=599 height=641
src="data_classification_qsar_files/image004.gif" v:shapes="_x0000_i1025"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>Figure 2.</span></b><span style='font-size:12.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>
Clustering of the 2DProp descriptors (taken from reference<a
name="_Ref188235746"></a><a style='mso-endnote-id:edn5' href="data_classification_qsar.htm#_edn5"
name="_ednref5" title=""><span style='mso-bookmark:_Ref188235746'><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[5]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188235746'></span>)<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The various types of
agglomerative method differ in the method used to calculate the distance
between merged points (clusters); in the simplest version the distance between
the centroids is applied. It is up to the user to cut the dendrogram at some
vertical position and to determine how many clusters are considered.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>PLS mapping</span></u><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'> A less frequent used but powerful application of
partial least squares (PLS) regression is the connected projection of X- and
Y-data. Assume <i>n</i> objects (chemical compounds) are each characterized by <i>m</i>
features (structural descriptors) and <i>k</i> properties (biological
activities). These data can be collected in two matrices, X for the features
and Y for the properties. The PLS approach uses X and Y together and generates
a projection plane in X-space that is influenced by the Y-data and a projection
plane in Y-space that is influenced by the X-data. An advantage of this connected
projection is that clustering, i.e. of biological activity data, is influenced
by structural data and <i>vice versa</i>; relationships between X and Y can be
extracted more efficiently<a style='mso-endnote-id:edn6' href="data_classification_qsar.htm#_edn6"
name="_ednref6" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[6]</span></span><![endif]></span></span></a>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";color:red;
mso-bidi-font-style:italic'>TASK 1.</span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'> A good example on multivariate data analysis can be found in the below
given paper. Please <b style='mso-bidi-font-weight:normal'>read</b> and <b
style='mso-bidi-font-weight:normal'>understand</b> it! (download PDF1)<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Olah, M.; Bologa, C.; Oprea, T.I.
An Automated PLS Search for biologically relevant QSAR descriptors,<i> J.
Comp.-Aided Mol. Design</i>, <b style='mso-bidi-font-weight:normal'>2004</b>, <i>18</i>,
437-449.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.1.3 Multivariate calibration</span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The aim of this method is the
development of a mathematical model that describes the relationship between a
set of <i>x</i>-features and one or several <i>y</i>-variables. There are
several known techniques, such as (i) <i>ordinary least-squares regression</i>
(OLS), complemented by (ii) <i>principal component regression</i> (PCR), (iii) <i>partial
least squares regression</i> (PLS) (see above), and (iv) neural networks. The
main applications in chemistry are quantitative analyses in infrared
spectroscopy (especially NIR), evaluation of multi-sensor data, and
investigations of structure-activity/property relationships (QSAR/QSPR).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Two topics of multiple linear regressions
(MLR) are especially essential in chemometrics:<o:p></o:p></span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l0 level1 lfo7'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>The
     <i>x</i>-features are often highly inter-correlated. The idea is not to
     select a subset of less correlated features but rather to consider highly
     correlated features as multiple measurements that increase the stability
     of the model. Regression methods insensitive to correlated features are
     therefore to be applied.<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l0 level1 lfo7'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>The
     complexity of the model is optimized to give the best prediction of the <i>y</i>-variable
     but not for best fit of given data. In PCR and PLS the model complexity is
     controlled by the number of used components (intermediate latent
     variables). A <i>training set</i> of data is used to calculate and to
     optimize the model parameters; a <i>test set</i> (<i>prediction set</i>)
     of data – not used in model generation – is applied to test the model. If
     the data set is small a <i>cross validation</i> procedure is used to
     estimate the predictive power of the model.<o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Ordinary least-squares regression
(OLS)</span></u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'> is a method of multiple linear
regression that directly uses the features <i>x</i> to determine a final latent
variable with maximum correlation coefficient with <i>y</i>. This criterion is
equivalent to a minimum sum of squared residuals. OLS does not necessarily give
the best model for prediction and is sensitive to outliers.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Principal component regression
(PCR)</span></u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'> models the dependent variable <i>y</i>
as a linear combination of PCA scores. By definition, the PCA scores are
uncorrelated and the regression model can be optimized for best prediction by
varying the number of PCA components used. However, the optimum number of PCA
used is determined by crossvalidation. The advantages of PCR are: (i) is
strictly defined, (ii) the regressor variables (PCA scores) are uncorrelated
and the directions of the corresponding intermediate latent variables are
orthogonal, (iii) the same regressor variables can be used for different <i>y</i>-properties,
and (iv) the regressor variables have maximum variance. A disadvantage I sthat
the regressor variables are computed without considering <i>y</i> and,
therefore, might not be optimum for building the model. The alternative method
PLS is, therefore, often preferred.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.1.4 Multivariate classification</span></b><span style='font-size:
12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Classification of objects to one
of pre-defined classes is called <i>pattern recognition</i> and was one of the
first applications of multivariate data analysis methods in chemistry<a
style='mso-endnote-id:edn7' href="data_classification_qsar.htm#_edn7" name="_ednref7" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[7]</span></span><![endif]></span></span></a><sup>,<a
style='mso-endnote-id:edn8' href="data_classification_qsar.htm#_edn8" name="_ednref8" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[8]</span></span><![endif]></span></span></a></sup>.
One of the typical applications is classification of chemical substances with
regard to biological activity. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>As previously discussed, a matrix
X contains features that characterize the objects, i.e., chemical compounds,
and a matrix Y (or a vector <i>y</i>) contains variables with the encoded class
membership of the objects. In binary classification, the objects are assigned
to one of two mutually exclusive classes (yes/no decision); in these cases
usually a single <i>y</i>-variable is used that for instance has the values -1
or +1 for class 1 or class 2, respectively. For more than two classes a
separate binary variable can be used for each class, with the value 1 if the
object belongs to this class and 0 otherwise.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>One group of multivariate
classification methods is based on the concept of linear latent variables. A
so-called <i>discriminant variable</i> is calculated that optimally separates
classes (in most cases, two) – see Figure 3. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><!--[if gte vml 1]><v:shape id="Object_x0020_2"
 o:spid="_x0000_i1027" type="#_x0000_t75" style='width:468pt;height:223.5pt;
 visibility:visible' o:gfxdata="UEsDBBQABgAIAAAAIQCTXr2S+wAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWxQ5dIISSdEHKEhAqBxjZk8RqMrY8JrS3x2nLggVFLC3r/f++plrvp1HMGNl5quWtKqVA
Mt466mv5vn0q7qXgBGRh9IS1PCDLdXN9VW0PAVlkmriWQ0rhQWs2A07Aygek/NP5OEHKz9jrAGYH
PepVWd5p4ykhpSItGbKpXrJAdBbFK8T0DFPu0TayxpVvvVE5SorHE7PU1hJCGJ2BlKX1TFZNXPiu
cwZVG3lzpG4WSjdVix18jEls9rnytDLiyH8E/lxQnO1VJo+lPLjAFxouG5/Nfltt/SdFnP87u83Y
G87fVvp4oOYLAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJlbHOk
kMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qRJVI2
sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY22Yi
Tra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1iOWA1
4Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAAIQA5
xHPQYAsAAHynAAAOAAAAZHJzL2Uyb0RvYy54bWzsXV1z27YSfb8z/Q8cvnZSCwA/PbE7jdPkdiaT
euL0vsMUZWlMkRqQtuX++u4CICkqkiDHThT1bh4cSiQg8HAJ7Nld7L7+dTkvvPtc1bOqPPPZLyPf
y8usGs/KmzP/r8/vXiW+VzeyHMuiKvMz/zGv/V/Pf/rP64fFac6raVWMc+VBJ2V9+rA486dNszg9
OamzaT6X9S/VIi/h5KRSc9nAR3VzMlbyAXqfFyd8NIpOHio1Xqgqy+savn1rTvrnuv/JJM+aPyeT
Om+84syHsTX6r9J/r/HvyflreXqj5GI6y+ww5FeMYi5nJfxo19Vb2UjvTs2+oquiym7z8YUs72UN
XRbZ6eo3doxF9vye5Wl5/14trhaXCkeefby/VN5srLEp5RwelkEHTtjLAKyTtVY3fQfLiZpjR9Vk
4i11L48dwvmy8TL4MmEiYGHsexmcE0mSBDyxvzKFB4XteBimYeB7cAHjqWAxby/43dEJjM2MAQ5W
xqUPcWBbbjfgvmfu972q7hZewJ5139vGf0gIsmV5tTAQXODh+hNnooXgqlFydjNtvN+Uqh68i6os
4R2qlMfax/Dx3nbRSsNKj7XtGR+Dpyp428JghP/0u2EFg7OYxfAVPmEe8SSFY/0atghFozTC8ygj
oRDQA57vnq48Xai6eZ9Xcw8PzvzajrkbLNM/J+8/1I1p2DZACIrSewDRSkehGRWeeyvrqXcvYY4Y
w5EZTSNnxe/l2GseF/AqSITDDqMo9Wjam62bxyI3PX/KJ/AGgRCbEehZLL8olOlbZlleNlq64GaK
Eq7GZpNZUXQNzZh2NrTXY9Ncz3BPady10L9clU3XeD4rK6WBW/v1ZtkOeWKuN3dv7hpupBOu7qBs
RQR/pJ9XOHNKGbeiAI2eK2WCB8xKGUsDgccDKWMgY7CCGDFjIsbPJGcrAnq8cuaezfjLzWZhOkqs
nHF7PJSzURTpC3A6S1lA09lwHjxaMROgxhi9YeuiKSIz5+w/nU2K2eK/7fphF0wRp6MUVitcMEXE
4HBtKkvCmON5lDA24gmurrRi9kvt8YpY6Bax4EVELIy4aEWM8QAXxuEsxiKOGpORMSHgYpKxVXXu
eGUMHqpjGuPpU2Vsq+6fpqB0wfqMM1Ua8TiNh3LG+SjtVkvQ21gy0nPot9L+66qYjd+BBo6q6pri
O1ynBlcSaTA85/mkoba89EtSCjYAI5efgIHK8qbIPdHJIV5ulrjyvm3aErI1QwRnQmg+iTIXBDGK
12BuC0c87ZiAGAVwgWNuUzCiXRyzrFCkzPjaUTVLQ8Gb5Ztq/Ijidg3/g8lFNcVFZcxTssymFVin
skaZIRZ1c9XSy3E+uTTEHY4+QcMCQDnz8/LVX1d4Nbwi/SXFfcEAIm8u1Qdti5HFDRjoCt+Daz7L
66u/O2UUB6AvyeWH8o26Re0D+m5mpf0IE8QUfgrMa5d3ZQaE27BbO4oaeoK3FC66zRXaABFK/Tbt
eLWKjgYPXiv9q5pxT2QGrPvnefmqaOzTkmsncmlOZPXaiazu4QCcNDIWDxwXHPIemgB0Jxw84YOg
WHxEj49lLIRPgaBYfIIen9ZyQAAViIoFKFwBKOGJnh7oDUNULEBRDxDnCSxPNAXBvIyoWIDiFYDi
QNAcrRcuRMUClPQAITpoVqI5qEBULEDpCkBRGGsVkOYgRMVoRCuqpdbBES6DT6d+5uX4Uiq5WdmE
RqBy9ursXZ1fLVBXN7poq+9uco9wH5/RFrqlT03lODdeEzA0gGhbXRrdwuhQMXew0YOyh+vle3P1
Xtvd7EExHMjlPolaPrTV3BhahXhv5wlaG//X6vPW2hhEQaJnW7Q2Bhy8c2t0qZtttLkxiqNvYtHm
6DnUYkKOOOs/fDFHHMRkOCw+1iqzv91aG3wYekJQXvAVtuLUMm7t7Q1EYoUFfJfWmw8O4FDPzShO
3C715nVviXzrvn2Gv5fE6Yv5dm9x2m6iSVtB+hOc514byfFRW2TMI3RaZ8CQHLbTDd8kHyzkWrfR
s4053ikdeQGTWp1rGdwSAjCwOdTq5rpz0b97p+XXrjarBgx0+gOEq1/t1RKWyLV4gXZh7FdOMgQB
SgNoyRBk5/xedfh++JAhyJoetxgSyRDU2ma3AESGIAdAZAhyAESGIAdAZAhyAESGIAdAZAjqQm9R
s+8CY41je4843K4FNreGHYdXvNfmdhmCtvMtBjZew9w14erDMq7AeLcv4YqF0P4YdIfHIUfuDU17
Qk6EC9Egzzt53ldCg74pISXCRYQLdpn0MS9PDW0hwuVQdohwOQAiwuUAiAiXAyAiXA6AiHAdIeHq
tixqwgX8S1Ml8Is+gXHxKIIdOUDd0MEZsoAYF27Fp1hniLBei1ymWOd+MwYxrgO6AMnF5VjMiXE5
ACLG5QCIGJcDIGJcDoCIcTkAIsZ1fIyLd9s+/6irOsuLvPY+q5nZ/8n33wBq0g/BXuMRi9JoNSI1
SEPR0jHWhy/3DjCzIcdsdbfHxre2JR61scP72pBDDDpEXvlFEJwz5HBzSwo5HAYTru0z6J2w3y+k
Dvfiaq8i7T21km52D5t9HAYUiw9SD7ttmfaetvgQH3Ms9cTHHAARH3MARHzMARDxMQdAxMccABEf
O0I+1qUg28TH+uRjLn9Yy8dCznkaDPiYzsBi3WNprHPJDAISiY9pOCgikSISKSIRQnN/AD5PfMyx
1BMfcwBEfMwBEPExB0DExxwAER9zAER87Aj5WJcGaBMf6zMA7cvHgjCBZM4DPgYOsnaDWJudg/gY
xStiRliKV4QcugfIXUs7xGiHGO0Q+5YzEPExh7JIfMwBEPExB0DExxwAER87Pj4mumSanyGj5Ztq
6UFdKE2X7CYxD/P86ZKA8C2E5jkzIrKEsVGwoQJmmzFTRDpph6mhGaVC6GJm0PWW+MQXrVbxALVS
z/wSirn6K5UrMHarXvx210DhC5v/1yQ1/Den0ugTaFIg4ab6OYfA55iI0iHwOSrH1SEAOirH1SEA
OiqidAiAjoooHQKgoyJKhwDoqIjSIQDaTpT0Lgu1VhzNG89UA8kXIMasnjcXRS6hTplN5dCcL3UZ
C62bm9aos37RxbWs82JWghnyFdRaxWwOOzvla51uqaTh7ejW0IW9qmzgziFdw2N76kDR5WrveMow
X/uTeUqUjqBSO9dpLQIoFJGGtqgy8ZQfq8zMId7QY9rwdAh8iKf8ixw6hxAg4ikOYyrxFAdAxFMc
ABFPcQBEPMUB0I/FU7RXojlnA2oC1OGlKEYApMhkJ+8oxjBD+ZMpRpwEkYgNxRCcwdYhXQy+T9UQ
BiJGZoPFoQS5QgJdKesA9bwPoQERxditQhPF2I0PuUIcqxdRDAdARDEcABHFcABEFMMBEFEMB0Av
SjEuL4bkABTtzRXAe+fHy3opgi7fdkshgFRo3vK10VQ8DNPQBFNxKE9sg6V6BsFEFEOGOEMhIqAb
gtkMVRRN5Xu5/FC+UbdYItzTCZntR2B6U0jJNytvLu/KrGlLiJukXl79N3yhtxDd5gpcbQwL/OoQ
sUH53LaYu6nzThRirQbu2jb3Q+BDFIIoxP/PtpNDvGFEIRz6DVEIB0BEIRwAEYVwALSdQmAqVhPL
kjXK6OGuAKtcKa0t1vN/AAAA//+Uj00OgyAQha9i2Dc2XRp1UT2A0fQAU5ggKQIZqNXbF2TT7trN
wPy8N9+ETiOYhp1Z2dZQhbZXnpNalAET6jJVUqTURCMGIBgHKjQY2TA0p9vECqEo7Mmi8MuXXxS6
JMwhykDLuIsHytsOV/rZLsGsQAruGv9Fi9dlmnTSdrViT2BPj5MbkYfcDtsUUeOEz48kdxQ0r7Tl
DxQdmBX8MSIJ3Kx4DwE+8/h/uQovdrZaILVvAAAA//8DAFBLAwQUAAYACAAAACEA2l4jQN0AAAAF
AQAADwAAAGRycy9kb3ducmV2LnhtbEyPzU7DMBCE70h9B2srcaMOULU0xKkKghMSor/i6MRLEhGv
U9ttA0/PwqVcRhrNaubbbN7bVhzRh8aRgutRAgKpdKahSsFm/Xx1ByJETUa3jlDBFwaY54OLTKfG
nWiJx1WsBJdQSLWCOsYulTKUNVodRq5D4uzDeasjW19J4/WJy20rb5JkIq1uiBdq3eFjjeXn6mAV
dPvFy1sh9zP/vly7uHt4+n7dbpS6HPaLexAR+3g+hl98RoecmQp3IBNEq4AfiX/K2ex2wrZQMB5P
E5B5Jv/T5z8AAAD//wMAUEsBAi0AFAAGAAgAAAAhAJNevZL7AAAA4QEAABMAAAAAAAAAAAAAAAAA
AAAAAFtDb250ZW50X1R5cGVzXS54bWxQSwECLQAUAAYACAAAACEAOP0h/9YAAACUAQAACwAAAAAA
AAAAAAAAAAAsAQAAX3JlbHMvLnJlbHNQSwECLQAUAAYACAAAACEAOcRz0GALAAB8pwAADgAAAAAA
AAAAAAAAAAArAgAAZHJzL2Uyb0RvYy54bWxQSwECLQAUAAYACAAAACEA2l4jQN0AAAAFAQAADwAA
AAAAAAAAAAAAAAC3DQAAZHJzL2Rvd25yZXYueG1sUEsFBgAAAAAEAAQA8wAAAMEOAAAAAA==
">
 <v:imagedata src="data_classification_qsar_files/image005.png" o:title=""
  croptop="-2945f" cropbottom="-168f" cropleft="-1117f" cropright="-1255f"/>
 <o:lock v:ext="edit" aspectratio="f"/>
</v:shape><![endif]--><![if !vml]><img width=624 height=298
src="data_classification_qsar_files/image006.gif" v:shapes="Object_x0020_2"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>Figure 3. </span></b><span style='font-size:12.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>A
discriminant variable is a latent variable that gives optimum separation of
object classes. Projection on to the first principal component, PC1, does not
separate the classes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The loading vector that defines
this latent variable is called <i>decision vector </i>or <i>classifier</i>. The
mathematical criterion applied is maximum discrimination of the scores for
objects from class 1and class 2, or optimum prediction of the <i>y</i>-variable
that indicates class membership. For this approach, the methods of multiple
regression are applied, i.e., linear discriminant analysis (LDA), PLS
discriminant analysis. However, non-linear discriminant variables can be
realized by neural networks<sup><span style='mso-field-code:" NOTEREF _Ref188270291 \\h &#1; \\* MERGEFORMAT "'>3<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200370030003200390031000000</w:data>
</xml><![endif]--></span></sup> or by support vector machines<a
style='mso-endnote-id:edn9' href="data_classification_qsar.htm#_edn9" name="_ednref9" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[9]</span></span><![endif]></span></span></a>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Another group of classification
methods is based on a modeling of the object classes and on distance
measurements such as classification by distance measurement to class
prototypes, and SIMCA – Soft/Simple Independent Modeling of Class Analogy<a
style='mso-endnote-id:edn10' href="data_classification_qsar.htm#_edn10" name="_ednref10" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[10]</span></span><![endif]></span></span></a><sup>,<a
name="_Ref188273706"></a><a style='mso-endnote-id:edn11' href="data_classification_qsar.htm#_edn11"
name="_ednref11" title=""><span style='mso-bookmark:_Ref188273706'><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[11]</span></span><![endif]></span></span></span></a></sup><span
style='mso-bookmark:_Ref188273706'></span>. The <i>k</i>-nearest neighbor
classification (KNN) avoids any generalization of the data and does not define
latent variables; a new object is classified by the majority of known objects
situated nearest to the unknown.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Classification by distance
measurement to class prototypes</span></u><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>A simple prototype of a class is
the class centroid which is defined by the class mean vector. In a first
approximation a new object is assigned to the class with the smallest Euclidean
distance to the class centroid. Points with identical Euclidean distances to
two class prototypes define a decision plane that is equivalent to the symmetry
plane between the centroids. In a more general approach, Mahalanobis distances
are used, considering the correlations between the features. Points with equal
Mahalanobis distances from a centroid define an ellipsoid with a shape similar
to the distribution of the class. Calculation of Mahalanobis distances requires
the inversion of the covariance matrix.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Soft/Simple Independent Modeling
of Class Analogy (SIMCA)</span></u><span style='font-size:12.0pt;line-height:
115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'> was
created by Svante Wold<sup><span style='mso-field-code:" NOTEREF _Ref188273706 \\h &#1; \\* MERGEFORMAT "'>11<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200370033003700300036000000</w:data>
</xml><![endif]--></span></sup>. Each class of objects is represented by a
separate PCA or PLS model. For each object the probability of its belonging to
a certain class or being an outlier can be estimated on the basis of <i>F</i>-test
statistics<a style='mso-endnote-id:edn12' href="data_classification_qsar.htm#_edn12" name="_ednref12"
title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA;mso-bidi-font-style:italic'>[12]</span></span><![endif]></span></span></a>.
Because of this flexibility SIMCA is called <i>soft</i>. Other classification
methods like discriminant analysis are called <i>hard</i> because a discrete
answer is given for class membership.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The simplest model of a class is
the class centroid that constitutes the center of a hypersphere. The radius can
be chosen so that 95% of the objects are situated within the hypersphere. More
complicated models use PCA components as axes of a hyperellipsoid; the optimum
number of components is determined by crossvalidation. Because SIMCA is based
on PCA, it is not sensitive to a large number of features or to collinear
features.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>KNN classification</span></u><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>An unknown object can be
classified by investigating the <i>k</i> nearest neighboring objects for which
the class memberships are known. To find the nearest neighbors of an object it
is necessary to compute the distances to all objects of a given data set. An
unknown object is assigned to the class to which the majority of the <i>k</i>
neighbors belong. Typical values for <i>k</i> are between 1 and 10.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The advantages of the KNN
classification method are mathematical simplicity, classes of objects need to
be linearly separable, easy application to multiclass problems, and no training
of classifiers is required. The disadvantage is the lack of data reduction
because a KNN classifier consists of all object vectors in the training set;
this results in large computation times for large data sets.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.2 Learning methods</span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.2.1 Supervised learning</span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Supervised techniques involve
learning from pre-classified training examples. Each training example is
identified as belonging to a particular class. Typically, the goal of such
methods is to derive a classification mechanism for correctly assigning new
examples to the set of classes represented in the original set of training
examples. The assumption is that the training data contained examples of all
possible classes so that post-training examples do not represent members of
previously unseen classes. The classification mechanism, whether it is in the
form of rules or a network, can be thought of as a function that maps a
description of an example to the name of the example’s class. The accuracy of
the classification mechanism can be determined with respect to a set of test
examples by applying it to those examples and checking to see if they are
properly classified.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Decision trees<o:p></o:p></span></u></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>Like most supervised
learning methods, the goal of the decision tree methodology is to develop
classification rules that determine the class of any object from the values of
the object’s attributes. In the case of decision trees, as the name implies,
the classification rules are embodied in knowledge representation formalism
called a decision tree<a style='mso-endnote-id:edn13' href="data_classification_qsar.htm#_edn13"
name="_ednref13" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[13]</span></span><![endif]></span></span></a>.
This method has been used to derive structure-activity relationships (SAR)<a
style='mso-endnote-id:edn14' href="data_classification_qsar.htm#_edn14" name="_ednref14" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[14]</span></span><![endif]></span></span></a>
and to learn classification rules for reactions.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><i style='mso-bidi-font-style:normal'><span style='font-size:12.0pt;
line-height:115%'>Decision </span></i><span style='font-size:12.0pt;line-height:
115%'>or <i style='mso-bidi-font-style:normal'>Classification Tree</i> analysis
is a hierarchical, highly flexible set of techniques for predicting membership
of objects or cases (i.e., chemicals) in the classes of a categorical dependent
variable (i.e., aggregator, nonaggregator) from their measurements on one or
more predictor variables (i.e., theoretical molecular descriptors,
fingerprints). This method of statistical analysis represents one of the main
techniques used in <i style='mso-bidi-font-style:normal'>data mining</i> and is
technically known as binary recursive partitioning; (i) binary, because parent
nodes are always split into exactly two child nodes, and (ii) recursive,
because the process can be repeated by treating each child node as a parent..
The key elements of decision tree analysis are a set of rules for: (i)
specification of the criteria for predictive accuracy, which is operationally
defined as the prediction with the minimum “costs”; the term “costs”
corresponds to the proportion of misclassified cases; (ii) selection of the
splits on the predictor variables, which are used to predict membership in the
classes of the dependent variables for the cases or objects in the analysis;
these splits are selected one at time, starting with the split at the root
node, and continuing with splits of resulting child nodes until splitting
stops, and the child nodes which have not been split become terminal nodes;
(iii) determination of the stop splitting, which is the user defined criteria
that allow splitting to continue until either all terminal nodes are pure or
contain no more than a specified minimum (a) number of cases or objects, or (b)
fraction of the sizes of one or more classes; and (iv) selection of the
“best-sized” tree is done from the sequence of optimally pruned trees based on
the cross-validation (CV) costs; usually, the “best-sized” tree is the
smallest-sized (least complex) tree whose CV costs do not differ appreciably
from the minimum CV costs.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>A decision tree used
for the classification of a set of chemicals as (i) aggregators (promiscuous
aggregating inhibitors), and (ii) nonaggregators is shown in Figure 4 (<i
style='mso-bidi-font-style:normal'>unpublished work</i>). According to McGovern
<i style='mso-bidi-font-style:normal'>et al.</i><a style='mso-endnote-id:edn15'
href="data_classification_qsar.htm#_edn15" name="_ednref15" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[15]</span></span><![endif]></span></span></a><sup>,<a
style='mso-endnote-id:edn16' href="data_classification_qsar.htm#_edn16" name="_ednref16" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[16]</span></span><![endif]></span></span></a></sup>
aggregators are nonspecific / non-lead-like small organic molecules that form
submicrometer aggregates (~30 – 400 nm), which are responsible for the
inhibition of many different enzymes through protein adsorption onto the
surface of aggregates, although absorption certainly cannot be excluded. In
fact, these molecules form large colloid-like aggregates that sequester and
thereby inhibit enzymes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><!--[if gte vml 1]><v:shape id="_x0000_i1028" type="#_x0000_t75"
 style='width:369pt;height:355.5pt'>
 <v:imagedata src="data_classification_qsar_files/image007.emz" o:title="2D_1D_v_3_June_9_picture"
  cropbottom="32975f" cropright="14838f"/>
</v:shape><![endif]--><![if !vml]><img width=492 height=474
src="data_classification_qsar_files/image008.gif" v:shapes="_x0000_i1028"><![endif]></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%'>Figure 4.</span></b><span style='font-size:12.0pt;line-height:
115%'> The classification trees with lowest relative cost developed using a set
of 1D and 2D descriptors. As can be seen, the major splitter (at root node) is
ALOGP (estimated value of the octanol/water partition coefficient). The second
splitter (at child node) is represented by one of the SMDL SMARTS – Q215,
[#7!H]~[#6R!H]<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>In general, many
different tree structures are possible. Shorter trees are usually preferred
because they express simpler classification rules that are easier to
understand.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;color:red'>TASK 2:</span></b><span style='font-size:12.0pt;
line-height:115%'> What is the 2D representation of the SMDL SMARTS Q215?<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Bayesian Belief Networks</span></u><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>An important consideration in
many learning scenarios is the reliability of data and/or missing values. One
learning method that is designed to reason in cases of uncertainty is that of
Bayesian probabilistic networks<a style='mso-endnote-id:edn17' href="data_classification_qsar.htm#_edn17"
name="_ednref17" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[17]</span></span><![endif]></span></span></a>. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>At a qualitative level, a
Bayesian belief network is a directed acyclic graph in which the vertices
represent domain objects. The links (arcs) between the vertices of the network
denote casual relations between the objects represented by the vertices. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Quantitatively, the casual
relations expressed between vertices by the arcs are represented by conditional
probabilities. For each vertex with parents there is a conditional probability
distribution. For each vertex without parents there is a prior-probability distribution.
These probabilities describe the uncertainty in the casual relationship between
the attributes represented by the vertices. When the prior and conditional
probabilities have been learned from pre-classified training examples, it is
possible to compute any probability of the form. In particular, given an
unclassified instance, it is possible to determine its classification with an
associated probability. In addition, the method still works even when values
for some of the attributes are missing. This method is ideal when complete
information is not available.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The approach used to calculate
missing values or to determine the propagation of effects caused by changing or
setting the value of a specific variable in a network is that of probabilistic
propagation in trees. A hallmark of this method is that the probabilities can
be computed solely on the basis of communication between neighboring vertices
in which the intervertex communication occurs. </span><span style='font-size:
12.0pt;line-height:115%'>In bioinformatics, Bayesian belief networks are used
for modeling knowledge in gene regulatory networks, protein structure
information etc.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%'>Support Vector
Learning</span></u><span style='font-size:12.0pt;line-height:115%'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>The Support Vector
Machine (SVM) belongs to the class of kernel-based learning methods. A key idea
underlying SVMs is the selection of features from a very high dimensional
feature space and the use of a simple linear hyper-plane decision surface to
effect a classification. SVMs provide a new approach to the problem of learning
with a clear connection to statistical learning theory.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>Statistical learning
theory shows that for a given number of training examples, there is a trade-off
between the degree to which the empirical error can be minimized and to which
the empirical error will deviate from the true error. It has been shown that
the capacity for generalization is related to the complexity of the
classification functions from which the learning machine can choose<a
style='mso-endnote-id:edn18' href="data_classification_qsar.htm#_edn18" name="_ednref18" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[18]</span></span><![endif]></span></span></a><sup>,<a
style='mso-endnote-id:edn19' href="data_classification_qsar.htm#_edn19" name="_ednref19" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[19]</span></span><![endif]></span></span></a></sup>.
Essentially, the theory says that a “simple” classifier that explains most of
the training data is preferable to a complex one (Occam’s razor). SVMs have
been successfully applied in computational chemistry<a style='mso-endnote-id:
edn20' href="data_classification_qsar.htm#_edn20" name="_ednref20" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[20]</span></span><![endif]></span></span></a><sup>,<a
style='mso-endnote-id:edn21' href="data_classification_qsar.htm#_edn21" name="_ednref21" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[21]</span></span><![endif]></span></span></a>,<a
style='mso-endnote-id:edn22' href="data_classification_qsar.htm#_edn22" name="_ednref22" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[22]</span></span><![endif]></span></span></a></sup>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%'>Genetic Algorithms
(GA)</span></u><span style='font-size:12.0pt;line-height:115%'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>The area of genetic
algorithms has been inspired by ideas from biology such as genetic
recombination, natural selection, and random mutation. Under the genetic
algorithm paradigm, objects, typically solutions to problems are represented as
bit strings. A fitness function is used to measure the goodness of solutions.
The genetic operators are then applied to objects to generate new objects in
proportion to their fitness. The goal is to find objects with maximum fitness. </span><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>1.2.2 Unsupervised learning</span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The problem of unsupervised
learning is more difficult than that of supervised learning. The task involves
looking for regularities in training examples that have not been pre-classified.
Because of that, part of the learning task entails deriving classifications.
Typically, such learning systems implement a bias to restrict the hypothesis
space and to form a preference among hypotheses. The motivation is to restrict
hypotheses from what might be combinatorially possible to those that are
conceptually meaningful and then rank them according to the utility of the
concepts they express.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Unsupervised concept learning can
be accomplished by first analyzing the unclassified training examples and
forming a taxonomy. Next, the descriptions of the examples in the resulting
classes can be generalized to form class descriptions expressing the concepts
embodied by the members of each class.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The basic taxonomy task is that
of classifying a set of objects represented by attribute-value pairs. This task
can also be viewed as a two-part process, first identifying the classes represented
by the original set of objects, and then classifying those objects according to
the identified classes.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><u><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Artificial Neural Networks (ANN)</span></u><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>ANN are a method or, more
precisely, set of methods that has recently found very intensive use among chemists.
ANN are not one method, but several different computational methods featuring a
wide variety of formal layouts, learning strategies, and sought goals. In the context
of ANN the term <i>layout</i> means a <i>set-up</i> or <i>architecture</i> of
artificial neurons forming the network; <i>learning strategy</i> means a
mathematical procedure and conditions put on to the data set to obtain the
final (the <i>trained</i>) ANN. By the term <i>sought goal</i> the final result
as produced by the trained ANN, like <i>mapping</i>, <i>classification</i>, <i>model</i>,
etc., is understood.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>ANN are difficult to summarize
within a simple definition. Maybe the closest definition would be comparison
with a black box having multivariate input X = (x<sub>1</sub>, x<sub>2</sub>, x<sub>3</sub>,
…, x<sub>m</sub>) and multi-response output Y = (y<sub>1</sub>, y<sub>2</sub>,
y<sub>3</sub>, …, y<sub>n</sub>) – see Figure 5.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><!--[if gte vml 1]><v:shape id="Object_x0020_3"
 o:spid="_x0000_i1029" type="#_x0000_t75" style='width:468pt;height:252pt;
 visibility:visible' o:gfxdata="UEsDBBQABgAIAAAAIQCTXr2S+wAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWxQ5dIISSdEHKEhAqBxjZk8RqMrY8JrS3x2nLggVFLC3r/f++plrvp1HMGNl5quWtKqVA
Mt466mv5vn0q7qXgBGRh9IS1PCDLdXN9VW0PAVlkmriWQ0rhQWs2A07Aygek/NP5OEHKz9jrAGYH
PepVWd5p4ykhpSItGbKpXrJAdBbFK8T0DFPu0TayxpVvvVE5SorHE7PU1hJCGJ2BlKX1TFZNXPiu
cwZVG3lzpG4WSjdVix18jEls9rnytDLiyH8E/lxQnO1VJo+lPLjAFxouG5/Nfltt/SdFnP87u83Y
G87fVvp4oOYLAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJlbHOk
kMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qRJVI2
sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY22Yi
Tra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1iOWA1
4Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAAIQCg
S+3qlA4AAA0EAQAOAAAAZHJzL2Uyb0RvYy54bWzsndty4kYQhu9TlXdQ6TbltaTR0bU4lXWye7O1
cS2bBxgLYVMrJEqSMc7Tp+ckGRloA3YAV9/Y2EiD+NWa6W+6p+fj74tpbs2zqp6UxcB2Pzi2lRVp
OZoUtwP7nx+fz2LbqhtejHheFtnAfsxq+/fLX3/5+DC7yLzyrsxHWWVBI0V98TAb2HdNM7s4P6/T
u2zK6w/lLCvgzXFZTXkDf1a356OKP0Dr0/zcc5zw/KGsRrOqTLO6hv/+qd60L2X743GWNn+Px3XW
WPnAhmtr5M9K/rwRP88vP/KL24rP7iapvgy+w1VM+aSAD22b+pM33LqvJjs0lZfpz2x0xYs5r6HJ
PL14+h99jXm6f8v8oph/qWbD2XUlrjz9Nr+urMlIalPwKdwspQ68oQ8Dsc57Z912DSzG1VQ0VI7H
1kK28tgqnC0aK4V/RpEbJFFgWym857uBn3ih/pQ7uFHiPD+I4MbaFhzgMjeJnfaAv5BG4NrUNcCL
J9clX4oLW/N1fbgc9X2/VOX9zPL9vb73uus/pAT1TH3/5/faN9/9OzwqvLjNM4u1X18cbu65ObXW
5tK728zxY8fcNt+P4A/VjPnaHgs9eV/FnWe+l4ijVeOmqVlVN1+ycmqJFwO7giuSzxSff60bdag5
RHyduswno8+TPJd/iA4ju8ora87hUW8Wrm586ai82OlEsCdxJvwy375ZDKWkzeJTOXoUjd7Ab3h+
qia/KlVfw4v0roSuJm0qJUVeN8PmMc/E4aNsfK2eO3j1HU7MQfyBnRVn/wz1lXeH5PPcBdmtKa++
ygeL57fQ2+a2Bcf84DfDfwd24vq+0B8uQB6S8a/Fp+onPEQ2tN1MCv0nHHIHHwV95fV9kYLMrtRY
X0UNLbni3lk/s0p06OKWYZrlzWqxxadazeMsG/MUepPfpsVZ3igpMt57I+PqjbTuvZHWnRygk7wL
Wg9xXfDS66Qxzx7pI0TR+rBOH2MkpI8QRevjd/q4LHJDYf0kkFBFCxQ8ESj2Ytk9kEBCFS1Q2Ank
eTEYEFkQ9MtCFS1Q9ESgyGfSDyALEqpogeJOIKGOHMlJIKGKFih5IlAYRNRJS9dHqKI8oieupXRM
hVzKgFr3U0pZ9VxNazSpGslpVj1trvKMg9enXffm8lPO05/WTbkQvm8jP0m2khWja17xvt/aNaa9
dXlG5yPf19lwJkBDvW2c6No4xXnxPRsDggIFera48XIaoPPqeZpmBTib8q07PsqUsx9Ah2tYoj1D
ySIaFEePARPatnUDy8Rg2laXpo8Xp2ZyFqE9WbvDm05uz5CfXBZNe/J0UpTVqm/WudBjdby8fC2M
5A7RVrooFHYU8yvxUj0dHbiHBuaGTcUnt3eN9UdVlQ/WVVkUoHpZWYH2sr+ZBtTXXWrPMI7BMs3z
Lot9QAMF5gELYkPuhvCE59QSnhvEsXbcTTuG3TTe1foa24tTt2UN7Clya/gk/6sYSUQY2Fx8Of0h
PT5bZVIvuO3SAJ/Zywtu+f9tLx3frraX1lLaF0u3uDOZCDUZM/2yk8kkfsA8aTKew3w/lDYBz5Se
ySGTkb1T22Govu8F9tae8fIu5tVMBmZ01XTZ2l4m2qeXSaAbgY4MxiQviCLo21VjZDLtKCbueWsA
J2EyCWoyumOAjkmPbNsMTEkEk4/SZBiMUkCoZDJmRu5kByYXbijSzSR7dTOOE6luhoWJA3EGspnT
txmY60ZsBswKehYZ7dqhowGfN1LujO9B8CrSkSIam054bHLhhmJGI5+MXY0mYY4HYTZwaPzICTxG
PU0bLzrd0YnhRuPt3tMEfhzKOUAwGjcIgth0W9TTnHJP0wbb15KT2wXftx6ewGi8EPwYiU5OCPSk
0Z2M5pSNps1OWW80XcLKLkbT9jReIIyGhqd3MDzhM8HuHlPB4PkCZhvidh3Xp57mHRgNPhfs7jEZ
DMNTFAMxieEJ0ix8PyCjeQdGg88Gu3tMBwcQMgghIVPSk+fEERmNCIAeJkj5ahEEF58PdrUbssuE
8JLRQMJx4GkUI0f4iB3h9TnKwtNQMzQ/4A5+KhcWGJCZxRNRcEukGEDSgHkwirmJja+LZUOIKY7V
TN5SkrmxEEhWlikmMlk5TBgEMdUztyaYjeUqF+VnyEBQTZhrMukPXX6EziF+gOUIA7uA9RIynVfn
E4tZy3r2x30DTekECnW8eOO9phZ3Xc5yHjelFm/Oc39LfU4ptfgQ9sN0Xg6kQB5/avEhBDqp1OJD
CHRSqcWHEOikUosPIdBJpRYfQqCTSi0+hEDrU4tlBvB2ecT9/OHVTdzwOssnYpXwmSdyemF526bk
ZPeFSckbmlUOOSRMi/TmdinfyjTlNgl3A6e06QeGU4Bc9uUUxwV6FdMfrl56IxskTjmu1RGHeELF
klDJfiewBPIQ+hCn6EW3a5bQEqeYVclrBCJOQQQiTkEEIk5BBCJOQQQiTkEEOi5Okc55c+ktoQmg
w6shRpus2iJGl5s63DEUwhKmIqweiyADqJfUTKGQ46iycggXmhBjswtNiLFZH0IMZPQixEAEIsRA
BCLEQAQixEAEIsRABDpKxGBvhhjt0qYWMbqVTDsihgeZvyqK4UUAGKbCpolieKEbivW+sigovA5l
bjpgE2VbQTnd/6+QIyHGblVFKdvqcNlohBjI6EWIgQhEiIEIRIiBCESIgQhEiIEItC1iiDrdTK5Q
2JzdJKo2fngzUmjXs7ek0C1f35UUQlcU5JBrBOMoivRCDyIFyneiYMTmyXYKRmzWh0gBGYSIFBCB
iBQQgYgUEIGIFBCBiBQQgU6TFNoiRi0pdDWLdiWFNqbAvBgq0vTqFlFMgdKWTmFzqEPEXIgUiBSW
tmXbdvc1IgVklCZSQAQiUkAEIlJABCJSQAQ6TVJoK1e2pNAVqtyJFJLEkYESCCn4fhI825jWd2D3
CIhkUKmnw+4iewhHmEIKmx1hAoXN+lBIARmDCBQQgQgUEIEIFBCBCBQQgQgUEIG2BYWuKlNWVaLq
6qqtY/cv+aQb10uqp0tZTLAB3potaDfnQ73iUmyxZnq5Ki3bM/spAhKBHVRVtSfGkijp7X3IQjgA
Cm4TqhCqTHn1FYg2iI5wM/VDoByhCqEKxTTeciUSoQriRxGqIAIRqiACEaogAu2OKvW0ucozXgxs
7VQ3l489pHifVWnZs9wrIJe9qtICpzgMNmhRVWlhH6f+em7iFMq9otyrWm8ZM8rG30WlXph70KLo
qr0iemApjktgPzRR8JpW+VBIBRkCKaSCCEScgghEnIIIRJyCCEScggh0XJzy1lVp2bOkLYCOfRED
6kRBs5C15TEoGEWIIef7yUUEF5qytjZP9VMoZLM+hBjI6EWIgQhEiIEIRIiBCESIgQhEiIEIdFyI
sSoFbLlE7TGkbEHMopeyFe3JKa4b+6J0rShY5SU+C0LVIBWsoqls4pTNfjhxymZ9iFOQIZA4BRGI
OAURiDgFEYg4BRGIOAURaFtOOYrStix5Rgq6vlT6bT7cZZ+9yHWDSARKZGnbxPP6EQ0qWEVJU5Q0
RUlTv02Ls23rMREpIIMQkQIiEJECIhCRAiIQkQIiEJECItBJkoIPk/+9mILes2JnUnBg3wtPbYLh
+7ETm/UaJqbAwsQRm2TQMnBaBk7LwGl5hVhOktZWs7T8OV1NUkQKyCBEpIAIRKSACESkgAhEpIAI
RKSACLQtKbyoYtX+y8GXK1YVveXlB69Y5UOlrmVUAXjZb5mG6yQB0zt7h3GSwGvZoEEVCmpQUIOC
GqtdcUp/ovSnJWTbNupDqIK4CYQqiECEKohAhCqIQIQqiEDbospRpD/54NL3SMHdjxRMoVLIfgpc
JxDFbpZAwQ2ZE0RQqoqCGhTUoKAGBTUoqFGc5Y3uJHkvuEOkMLBFntyrCUSkgLgxRAqIQEQKiEBE
CohA25JCF9RYUdt2etaLPqwubtu1sWErjzmvJrzJtmxwxUVZk2J23/TaWRcWaS8NOOEVt+3w2TO0
0TGLXfO1mOeHkUCXtWyTMD/2iG34zZDYhtiG2IbY5hVdd4qCIH4Fsc1zgf4DAAD//+xX227bMAz9
FUGvQ9H4lhvmAmuG7aUogqb7ANVmEqOybEiK6+zrR0oy4gXdgALbm18SOWQO6SOS0oH901bffb4V
a9nJeUtrt1zgktVCP+Q8XqRJPJtxJuRB5VxyVsL+WbzsfuZ8FaUpmbSVOcdvEA/qXr/mPOJMClup
8Iimo1CHSh22J1VYslMgBMLwzCBStCScV9AYIqJwZDeNrMpvlZTuoThCDRupWScwmu0jfusy/83L
RWX23MJeFJDzT7W6kZY8xRrElQGENxTmylCYgO0zHAgiVggIuVpeCCJ2HAsTQcRKIGg1ImieLeZT
BVHZECu+mozd2bMEYqulD8ebxm6Q2Cg5B3XzY4etVml7dq1laruRILA9Zr5o7Z1qsLYrBUIzDVT4
jaJOti6AwwNVboUW1GTvw/oOwgTof/19U54pl5OBXfsEhfVm2++8h3GZ+k/V7cJmF48d4ldlztOU
MyVqbLtn6O1907M0CR326NwZxeip/T3yBcQEsH6va0qh2e8ZOs7TbBUTLJKQzZZRls09IOKzgpAW
UbJKcNwU6JHMV0kSB+wBqdXGfoemZrTIucbX4hRBdA8mvODgQj+rhgaOT29IyhMwZugFmcKXftOi
zTnuA7gZuGncGCQY0345WYQKEbw/GeRo43G6bH3DhEk43iSfwsUFqydCksJUxmE5jRviw3HayfhC
TZotpvPKHVJESuAnufAzHNpT/RApgZ/0wk+ULKLpuHIFRKwEgrIRQct46a5rUwURK4Eguj0PV+Z4
iQU0jehJU4xUyKQpvBr7qOiaNMWgY/9QQf9WU9xcSQg33v6iS0Brus+zd/RJJ3QlLHwQ8B0g1pxs
e7JXQP9B3KAKOujWqR1ZrGVTvEK5EaoTximkA173j1XxVVgxfsb1W7uGuDk2sgR99wsAAP//AwBQ
SwMEFAAGAAgAAAAhAFFbm4nbAAAABQEAAA8AAABkcnMvZG93bnJldi54bWxMj0FLAzEQhe+C/yGM
4M0mrlrsdrNFKqIgBVvbe7oZdxeTSdik7frvHb3o5cHjDe99Uy1G78QRh9QH0nA9USCQmmB7ajVs
35+u7kGkbMgaFwg1fGGCRX1+VpnShhOt8bjJreASSqXR0OUcSylT06E3aRIiEmcfYfAmsx1aaQdz
4nLvZKHUVHrTEy90JuKyw+Zzc/Aadq/FysqXvJw9r99C7AvnHuNO68uL8WEOIuOY/47hB5/RoWam
fTiQTcJp4Efyr3I2u5my3Wu4U7cKZF3J//T1NwAAAP//AwBQSwECLQAUAAYACAAAACEAk169kvsA
AADhAQAAEwAAAAAAAAAAAAAAAAAAAAAAW0NvbnRlbnRfVHlwZXNdLnhtbFBLAQItABQABgAIAAAA
IQA4/SH/1gAAAJQBAAALAAAAAAAAAAAAAAAAACwBAABfcmVscy8ucmVsc1BLAQItABQABgAIAAAA
IQCgS+3qlA4AAA0EAQAOAAAAAAAAAAAAAAAAACsCAABkcnMvZTJvRG9jLnhtbFBLAQItABQABgAI
AAAAIQBRW5uJ2wAAAAUBAAAPAAAAAAAAAAAAAAAAAOsQAABkcnMvZG93bnJldi54bWxQSwUGAAAA
AAQABADzAAAA8xEAAAAA
">
 <v:imagedata src="data_classification_qsar_files/image009.png" o:title=""
  croptop="-2230f" cropbottom="-1176f" cropleft="-1181f" cropright="-1116f"/>
 <o:lock v:ext="edit" aspectratio="f"/>
</v:shape><![endif]--><![if !vml]><img width=624 height=336
src="data_classification_qsar_files/image010.gif" v:shapes="Object_x0020_3"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>Figure 5. </span></b><span style='font-size:12.0pt;line-height:115%;
mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>Artificial
neural network (ANN) as a black box. Multivariate input is transformed in a
non-linear manner to the n-response output<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>ANN methods can only be used if a
comparably large set of multivariate data is available which enables one to
train an ANN by example. Note that the ANN methods work best if they are
dealing with non-linear relationship between complex inputs and outputs.
Although in principle ANN can be used to model one-to-one input/output
relations this is rarely done in practice. To make an ANN operational it must
be trained with a known set of data. Even ANN can model linear relationships
also, the final result might often be worse than results obtained by simpler
standard statistical techniques. The first thing to be considered when
employing an ANN is the nature of the problem: does it require a supervised or
an unsupervised approach?<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>In general, problems associated
with data handling at an early stage require unsupervised methods. Only further
on in the later steps of data handling, after one became more familiar with the
measurement space, with the distribution of input variables, and with the
behavior of the responses, can one select sets of data on which supervised
methods (modeling for example) can be conducted.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The basic types of problems in
chemistry for which ANN can be used are:<o:p></o:p></span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l1 level1 lfo9'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>Selecting
     samples from among a large number of samples for further handling
     (sampling);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l1 level1 lfo9'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>Assigning
     an unknown sample to a specific class of a pre-defined (known in advance)
     number of classes (classification);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l1 level1 lfo9'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>Finding
     the inner structure of relationships between samples within the given
     measurement space (clustering of objects);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l1 level1 lfo9'><span style='font-size:12.0pt;line-height:
     115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:italic'>Making
     direct and inverse models for quantitative prediction of behavior, time
     changes, or effects of unknown or not-yet seen samples (modeling).<o:p></o:p></span></li>
</ul>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>2. Quantitative Structure-Activity Relationships (QSAR) – 3D QSAR<o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The underlying theory of
Quantitative Structure-Activity/Property Relationship (QSAR/QSPR) analysis is
that biological activity/physicochemical property is directly related to
molecular structure. Therefore molecules with similar structure will possess
similar bioactivities for similar proteins/receptors/enzymes and the changes in
structure will be represented through the changes in the
bioactivities/physicochemical properties.<o:p></o:p></span></p>

<p style='text-align:justify;line-height:115%'><span style='mso-bidi-font-style:
italic'>The aim of QSAR/QSPR is to find a quantitatively correlation between
the structure of compounds and their biological activity or physicochemical
properties and derive a model to predict the activity or properties of novel
compounds. </span>For example, biological activity can be expressed
quantitatively as in the concentration of a substance required to give a
certain biological response. Additionally, when physiochemical properties or
structures are expressed by numbers, one can form a mathematical relationship,
or quantitative structure-activity relationship, between the two. The
mathematical expression can then be used to predict the biological response of
other chemical structures. A<span style='mso-bidi-font-style:italic'> model
usually consists of a linear or multilinear combination of features,
descriptors, and coefficients.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The most general mathematical
form of a QSAR/QSPR model is:<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:14.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>Activity / Property = <b
style='mso-bidi-font-weight:normal'><i>f </i></b>(Structural Descriptors) <o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>As it is possible to generate a
large number of descriptors for each compound, the selection of features that
yield a reliable relationship is a complex and time-consuming task. Nowadays,
there are several methods widely used in this area.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The <i>linear or nonlinear
multiple regression analysis</i> (MLR) is one of the primarily methods used as
a statistical tool to derive quantitative models, and to check the significance
of these models and of each individual term in the regression equation (a brief
mathematical description of the method can be found on PDF2). Other statistical
methods, such as multivariate data analysis (principal component analysis, PCA,
or partial least squares, PLS, analysis), and discriminant analysis are
alternatives to regression analysis (see above).<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>2.1 QSAR/QSPR – general algorithm<a name="_Ref188298858"></a><a
style='mso-endnote-id:edn23' href="data_classification_qsar.htm#_edn23" name="_ednref23" title=""><span
style='mso-bookmark:_Ref188298858'><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><b style='mso-bidi-font-weight:normal'><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:"Times New Roman";mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[23]</span></b></span><![endif]></span></span></span></a></span></b><span
style='mso-bookmark:_Ref188298858'></span><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>The QSAR/QSPR model development and validation involves
several major steps, as presented in Figure 6:<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0in' start=1 type=i>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>decision</span></i><span
     style='font-size:12.0pt;line-height:115%'><span style='mso-tab-count:1'>           </span><span
     style='mso-tab-count:1'>            </span>(of the property to be
     modeled);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>construction</span></i><span
     style='font-size:12.0pt;line-height:115%'> <span style='mso-tab-count:
     1'>    </span><span style='mso-tab-count:1'>            </span>(of the
     data set);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>generation</span></i><span
     style='font-size:12.0pt;line-height:115%'> <span style='mso-tab-count:
     1'>      </span><span style='mso-tab-count:1'>            </span>(of the
     molecular structures);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>feature selection</span></i><span
     style='font-size:12.0pt;line-height:115%'> <span style='mso-tab-count:
     1'>         </span>(of the most informative subset of descriptors);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>development</span></i><span
     style='font-size:12.0pt;line-height:115%'><span style='mso-tab-count:1'>    </span><span
     style='mso-tab-count:1'>            </span>(of the QSPR model);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>interpretation</span></i><span
     style='font-size:12.0pt;line-height:115%'> <span style='mso-tab-count:
     1'>  </span><span style='mso-tab-count:1'>            </span>(of the
     model);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>validation</span></i><span
     style='font-size:12.0pt;line-height:115%'> <span style='mso-tab-count:
     1'>       </span><span style='mso-tab-count:1'>            </span>(of the
     model);<o:p></o:p></span></li>
 <li class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
     justify;mso-list:l6 level1 lfo10;tab-stops:list .5in'><i><span
     style='font-size:12.0pt;line-height:115%'>prediction</span></i><span
     style='font-size:12.0pt;line-height:115%'> <span style='mso-tab-count:
     1'>       </span><span style='mso-tab-count:1'>            </span>(of the
     property of interest).<o:p></o:p></span></li>
</ol>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='mso-bidi-font-size:12.0pt;line-height:115%'><!--[if gte vml 1]><v:shape
 id="_x0000_i1030" type="#_x0000_t75" style='width:540pt;height:406.5pt'>
 <v:imagedata src="data_classification_qsar_files/image011.emz" o:title=""/>
</v:shape><![endif]--><![if !vml]><img width=720 height=542
src="data_classification_qsar_files/image012.gif" v:shapes="_x0000_i1030"><![endif]></span><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-no-proof:yes'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-no-proof:yes'>Figure
6.</span></b><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-no-proof:yes'> Flow chart of a QSAR/QSPR model generation
(taken from reference<sup><span style='mso-field-code:" NOTEREF _Ref188298858 \\h &#1; \\* MERGEFORMAT "'>23<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200390038003800350038000000</w:data>
</xml><![endif]--></span></sup>)<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-no-proof:yes'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A<i> <b style='mso-bidi-font-weight:normal'>Decision</b></i>
implies choosing of the property to be studied that is highly dependent on the
availability of experimental data. A large amount of heterogeneous experimental
values lead to more stable and valid QSAR models.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A<i> <b style='mso-bidi-font-weight:normal'>Construction</b></i>
represents the assembling of the data set that consists of 2D and 3D molecular
structures obtained by molecular modeling (the three-dimensional rendering of
molecules in an environment produced by computer graphics). The molecular
modeling, as part of the computational chemistry, includes the so-called
computational methods by which the spatial coordinates of the atoms in
molecules and the respective interconnecting bonds are produced. These
computational methods can be divided into two major groups: (i) molecular
mechanics methods, and (ii) quantum mechanics/quantum chemical methods. Group
(ii) is usually subdivided into <i>ab initio</i> and semi-empirical methods<a
name="_Ref188299792"></a><a style='mso-endnote-id:edn24' href="data_classification_qsar.htm#_edn24"
name="_ednref24" title=""><span style='mso-bookmark:_Ref188299792'><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[24]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188299792'></span>. In molecular mechanics, the
potential energy of a given conformation of molecular systems is calculated as
a function of the coordinates of their atomic nuclei, which are treated as
Newtonian particles under the influence of a potential energy function or force
field. A force field is given by equation (1) together with the data
(parameters) required to describe the behavior of different kinds of atoms and
bonds. </span><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Arial Unicode MS"'><o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><i><span style='font-size:12.0pt;
line-height:115%'>E</span></i><span style='font-size:12.0pt;line-height:115%'>
= <i>E</i><sub>stretching</sub> + <i>E</i><sub>bending</sub> + <i>E</i><sub>torsion</sub>
+ <i>E</i><sub>non-bonded interaction </sub><span style='mso-tab-count:4'>                                          </span>(1)<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>where <i>E</i> – potential energy, <i>E</i><sub>stretching</sub>
– bond stretching energies, <i>E</i><sub>bending</sub> – angle bending
energies, <i>E</i><sub>torsion</sub> – torsional energies and <i>E</i><sub>non-bonded
interaction</sub> – energies of non-bonding interactions. The potential
functions generated by molecular mechanics are often useful for comparing
different configurations of a molecule.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>Most molecular mechanics methods use empirical data to
determine individual force constants (for example, bond stretch constants) and
equilibrium (&quot;strain-free&quot;) values for each geometrical
characteristics of a molecule (for example, bond lengths)<a name="_Ref188299820"></a><a
style='mso-endnote-id:edn25' href="data_classification_qsar.htm#_edn25" name="_ednref25" title=""><span
style='mso-bookmark:_Ref188299820'><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[25]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188299820'></span>.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>The quantum mechanical methods are based on Schrödinger
equation<sup>1</sup> in which the electrons are described by the respective
molecular wavefunctions. For molecular systems, the Schrödinger equation can
only be solved approximately.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>Typical <i>ab initio</i> electronic structure methods
consider the potential energy operator as time-independent and solve only the
spatial wavefunction. The a<i>b initio</i> calculations can be performed at the
Hartree-Fock level of approximation<sup><span style='mso-field-code:" NOTEREF _Ref188299792 \\h &#1; \\* MERGEFORMAT "'>24<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200390039003700390032000000</w:data>
</xml><![endif]--></span></sup> or using various post-Hartree-Fock theories
(configuration-interaction, multiconfiguration self-consistent field etc.).<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>Semi-empirical methods have been developed as approximations
to <i>ab initio</i> techniques that were fitted to experimental data (e.g.
structures and formation energies of organic molecules, spectroscopic data and
ionization energies). In this case, the Hartree-Fock self-consistent field
method is used to solve the Schrödinger equation with various approximations.
Semi-empirical methods are usually classified according to their treatment of
electron-electron interactions<sup><span style='mso-field-code:" NOTEREF _Ref188299820 \\h &#1; \\* MERGEFORMAT "'>25<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200390039003800320030000000</w:data>
</xml><![endif]--></span></sup>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>A<i> <b
style='mso-bidi-font-weight:normal'>Generation</b></i> is based on previous
presented computational methods (<i>ab initio</i>, semi-empirical) and involves
the calculation of the so-called molecular structure descriptors in order to
encode the compounds (see <i style='mso-bidi-font-style:normal'>Section III.</i>
<b style='mso-bidi-font-weight:normal'>Molecular descriptors and fingerprints</b>)<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A <b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'>Feature</i> s<i>election</i></b> of descriptors involves finding the
most informative subsets of descriptors from those in the descriptor pool due
to the fact that models with too many variables lead to bad predictions because
of over fitting. Various methods, like the classical forward selection,
backward elimination and stepwise regression<a name="_Ref188299912"></a><a
style='mso-endnote-id:edn26' href="data_classification_qsar.htm#_edn26" name="_ednref26" title=""><span
style='mso-bookmark:_Ref188299912'><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[26]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188299912'></span>, or, more recently, the genetic
algorithms (GA) can be used for the selection of the best combination of
descriptors. The genetic algorithms have been shown to be very effective in
performing descriptor selection in the case of large descriptor pools<sup><span
style='mso-field-code:" NOTEREF _Ref188299912 \\h &#1; \\* MERGEFORMAT "'>26<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003200390039003900310032000000</w:data>
</xml><![endif]--></span>,<a name="_Ref188300334"></a><a style='mso-endnote-id:
edn27' href="data_classification_qsar.htm#_edn27" name="_ednref27" title=""><span style='mso-bookmark:_Ref188300334'><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[27]</span></span><![endif]></span></span></span></a></sup><span
style='mso-bookmark:_Ref188300334'></span>.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A<i> <b style='mso-bidi-font-weight:normal'>Development</b></i>
involves the application of the descriptors using (i) multilinear regression
analysis, such as the Heuristic and best multilinear regression (BMLR) methods<a
style='mso-endnote-id:edn28' href="data_classification_qsar.htm#_edn28" name="_ednref28" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[28]</span></span><![endif]></span></span></a>,
(ii) multivariate statistical methods, such as principal component analysis
(PCA) and partial least squares (PLS)<a style='mso-endnote-id:edn29'
href="data_classification_qsar.htm#_edn29" name="_ednref29" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[29]</span></span><![endif]></span></span></a><sup>,<a
style='mso-endnote-id:edn30' href="data_classification_qsar.htm#_edn30" name="_ednref30" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[30]</span></span><![endif]></span></span></a>,<a
style='mso-endnote-id:edn31' href="data_classification_qsar.htm#_edn31" name="_ednref31" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[31]</span></span><![endif]></span></span></a></sup>
or (iii) non-linear methods, such as artificial neural networks<a
style='mso-endnote-id:edn32' href="data_classification_qsar.htm#_edn32" name="_ednref32" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[32]</span></span><![endif]></span></span></a>
(ANNs) to build mathematical models linking the descriptors directly to the
chemical property under investigation. <o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>The multilinear regression methods gave a linear regression
equation in which the property, Y, is a linear function of descriptor values,
X: Y = f(X). Several statistical characteristics give information about the
“goodness” of the model: R<sup>2</sup> – squared correlation coefficient, R<sup>2</sup><sub>CV</sub>
– squared cross-validated correlation coefficient, F – Fisher criterion value,
s<sup>2</sup> – squared standard error.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>In the principal component analysis (PCA), the original data
matrix is decomposed into new latent variables, and the variation among the
objects (e.g. organic compounds), is illustrated in score plots. The
corresponding loading plots describe the corresponding variation among the
descriptors. In the case of partial least squares (PLS), the latent information
in the independent parameters is related to the dependent variable (i.e.
property) through a weight vector. In both cases, the number of significant
descriptors is determined by a cross-validation procedure<a style='mso-endnote-id:
edn33' href="data_classification_qsar.htm#_edn33" name="_ednref33" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[33]</span></span><![endif]></span></span></a>
and the reliability and stability of the models are estimated by the following
statistical measures: the variation explained in the X-matrix (descriptors) (R<sup>2</sup>X),
the variation explained in Y (property) (R<sup>2</sup>Y), the cross-validated
explained variance (Q<sup>2</sup>), and the root-mean-square error of
estimation (RMSEE) and prediction (RMSEP)<a style='mso-endnote-id:edn34'
href="data_classification_qsar.htm#_edn34" name="_ednref34" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[34]</span></span><![endif]></span></span></a>.
<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A neural network is a non-linear method capable of modeling
extremely complex functional relationships. It consists of a process of
assimilation in the cognitive system similar to the neurological functions of
the brain and is capable to predict new observations from other observations
(modeling) after the executing of the so-called “learning process” from
existing data. During the last decades, by using diverse “training” algorithms<sup><span
style='mso-field-code:" NOTEREF _Ref188300334 \\h &#1; \\* MERGEFORMAT "'>27<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003300300030003300330034000000</w:data>
</xml><![endif]--></span> </sup>various types of neural network architectures
have been developed. Recent investigations involving computational neural
networks and genetic algorithms serve as examples of the application of the QSAR/QSPR
methods<a name="_Ref188300426"></a><a style='mso-endnote-id:edn35'
href="data_classification_qsar.htm#_edn35" name="_ednref35" title=""><span style='mso-bookmark:_Ref188300426'><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[35]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188300426'></span>. Three-layer, feed-forward neural
networks trained with a quasi-Newton method have provided excellent results in
several QSAR/QSPR studies<sup><span style='mso-field-code:" NOTEREF _Ref188300426 \\h &#1; \\* MERGEFORMAT "'>35<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003300300030003400320036000000</w:data>
</xml><![endif]--></span></sup>.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>An<i> <b style='mso-bidi-font-weight:normal'>Interpretation</b></i>
of the model implies (i) the explanation of how each of the descriptors from
the selected “best” subset contributes to the property of interest, (ii) the
assessment of reliable physicochemical meaning to descriptors, especially for
those provided by multivariate statistical analysis.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A<i> <b style='mso-bidi-font-weight:normal'>Validation</b></i>
of the developed models is an important aspect of any QSPR study. Once a
regression equation is obtained, it is important to determine its reliability
and significance. Several procedures are available to assist in this. These can
be used to check that the size of the model is appropriate for the data
available, as well as to provide some estimate of how well the model can
predict activity for new molecules. <o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>An Internal validation uses the data set from which the model
is derived and checks for the internal consistency. For the internal
validation, the parent data set can be divided into several subsets; e.g. the 1<sup>st</sup>,
4<sup>th</sup>, 7<sup>th</sup>, etc. entries go into the first subset (#1), the
2<sup>nd</sup>, 5<sup>th</sup>, 8<sup>th</sup>, etc. into the second subset
(#2), and the 3<sup>rd</sup>, 6<sup>th</sup>, 9<sup>th</sup>, etc. into the
third subset (#3). Then, three training sets <i>Set 1, Set 2 and Set 3 </i>are
prepared as combination of two subsets (#1 and #2), (#1 and #3), and (#2 and
#3), respectively. The corresponding test sets relate to remaining subsets (#3,
#2 and #1, respectively). For each training set, the correlation equation is
derived with the same descriptors, and the obtained equation is used to predict
the values of the property of interest for the compounds from the corresponding
test set<a style='mso-endnote-id:edn36' href="data_classification_qsar.htm#_edn36" name="_ednref36" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[36]</span></span><![endif]></span></span></a><sup>,<a
style='mso-endnote-id:edn37' href="data_classification_qsar.htm#_edn37" name="_ednref37" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[37]</span></span><![endif]></span></span></a></sup>.
<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>An External validation evaluates BEST how well the equation
generalizes the data. The original data set is divided into two groups – the
training set and the test set. The training set is used to derive a model that
further is used to predict the properties of the test set members, which were
NOT used to train the model. <o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>The Cross-validation repeats the regression many times on
subsets of the data. Usually, each molecule is left out in turn, and the R<sup>2</sup>
(cross-validated correlation coefficient) is computed using the predicted
values of the missing molecules (<i>leave-one-out</i>, LO). Another possibility
is to leave out more than one molecule at a time (i.e. 30%), though not all
possible combinations of molecules are used, a concession that makes the
computation tractable (<i>leave-many-out</i>, LM). However, if n molecules are
removed once from a total set of m, then nXm regressions are performed<a
style='mso-endnote-id:edn38' href="data_classification_qsar.htm#_edn38" name="_ednref38" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[38]</span></span><![endif]></span></span></a>.
</span><span style='font-size:12.0pt;line-height:115%;font-family:"Arial Unicode MS","sans-serif";
mso-hansi-font-family:"Times New Roman";mso-bidi-font-family:"Times New Roman"'><o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>In multivariate statistical analysis, cross-validation is
often used to determine the number of principal components retained in the
model – those that give the highest cross-validated R<sup>2</sup> are to be considered.
<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>The Monte Carlo cross-validation (MCCV)<a style='mso-endnote-id:
edn39' href="data_classification_qsar.htm#_edn39" name="_ednref39" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[39]</span></span><![endif]></span></span></a>
was developed as an asymptotically consistent method in determining the number
of components that can avoid an unnecessary large model and therefore decreases
the risk of over-fitting in the model.<o:p></o:p></span></p>

<p class=MsoNormal style='text-align:justify'><span style='font-size:12.0pt;
line-height:115%'>A<i> <b style='mso-bidi-font-weight:normal'>Prediction</b></i>
involves the use of the developed QSPR models to estimate the property of
interest for unknown compounds. If prediction is the main goal of the
regression, then a test set data must be reserved strictly for evaluation until
the equation is finalized: this reserved data cannot enter the regression
process in any form.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'>This general QSAR/QSPR
methodology can be applied, in principle, to any property that can be related
to the molecular structure, and it has been applied for the description and
prediction of a wide variety of chemical, physical and environmental properties
and activities<sup><span style='mso-field-code:" NOTEREF _Ref188300426 \\h &#1; \\* MERGEFORMAT "'>35<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003300300030003400320036000000</w:data>
</xml><![endif]--></span></sup>.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'>2.2 3D-QSAR methodology</span></b><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";mso-bidi-font-style:
italic'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>It is an improvement on “traditional’
(two-dimensional; 2D) QSAr by incorporating the molecules’ 3D structure,
steric, electrostatic, and interactive phrmacophore elements into building the
QSAR model. There are two sub-fields of 3D-QSAR: (i) receptor-independent (Ri)
methods construct QSAR models without the assistance of a 3D structure of the receptor,
and (ii) receptor-dependent (RD) methods use the 3D structure of the receptor
to aid in the construction of the QSAR model through the use of ligand-receptor
interaction descriptors.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>There are two major points of
contention common to all 3D-QSAR methodologies: the alignment and the
conformation of the molecules in a study. The most common reason a 3D-QSAR
model fails is that the non-bioactive conformation of the ligand is used in the
QSAR study. In 1980 Hopfinger<a style='mso-endnote-id:edn40' href="data_classification_qsar.htm#_edn40"
name="_ednref40" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[40]</span></span><![endif]></span></span></a>
included the additional pseudo-dimension of an ensemble of conformations to the
3D-QSAR paradigm, thus creating the field of nD-QSAR. This additional dimension
eliminated the question of which conformation to use in a QSAR study. A qualitative
3D-QSAR method, Probabilistic Receptor Potentials<a name="_Ref188304414"></a><a
style='mso-endnote-id:edn41' href="data_classification_qsar.htm#_edn41" name="_ednref41" title=""><span
style='mso-bookmark:_Ref188304414'><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[41]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188304414'></span>, calculates ligand atomic
preferences in the active site. The potentials for PRP are constructed by
fitting analytical functions to experimental properties of the substrates using
knowledge-based methods. Another advance in the field of QSAR was the
development of Virtual High-Throughput Screening (VHTS) methods<a
style='mso-endnote-id:edn42' href="data_classification_qsar.htm#_edn42" name="_ednref42" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[42]</span></span><![endif]></span></span></a> and
Virtual 3D-Pharmacophore Screening<a style='mso-endnote-id:edn43' href="data_classification_qsar.htm#_edn43"
name="_ednref43" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[43]</span></span><![endif]></span></span></a>.
These QSAR methods have been devised to assist in computer-aided drug design by
contributing different information about how the ligand interacts with its
potential receptor site.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'>The focus of 3D-QSAR is to
identify and characterize quantitatively the interactions between the ligand
and the receptor’s active site. The interactions between the 3D atomic
coordinates of the molecule and the receptor are correlated with the
bioactivities, producing a 3D-QSAR model. There are several methods of
achieving the creation of QSAR models based on the atomic coordinates. The most
widely known 3D-QSAR methodology is Comparative Molecular Field Analysis
(CoMFA)<a style='mso-endnote-id:edn44' href="data_classification_qsar.htm#_edn44" name="_ednref44" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[44]</span></span><![endif]></span></span></a>,
which is based on placing the ligands in a 3D grid and using a probe to map the
surface of the ligand on the basis of the ligand’s interaction with the probe.
Comparative Molecular Similarity Index Analysis (CoMSIA)<a style='mso-endnote-id:
edn45' href="data_classification_qsar.htm#_edn45" name="_ednref45" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[45]</span></span><![endif]></span></span></a> and
Self-Organizing Molecular Field Analysis (SOMFA)<a name="_Ref188303701"></a><a
style='mso-endnote-id:edn46' href="data_classification_qsar.htm#_edn46" name="_ednref46" title=""><span
style='mso-bookmark:_Ref188303701'><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[46]</span></span><![endif]></span></span></span></a><span
style='mso-bookmark:_Ref188303701'></span> are based on the same methodology as
CoMFA. CoMSIA works by separating and projecting where physicochemical
properties rationalize the binding affinity in 3D space by this method. Using
fundamental molecular properties and “mean centered activity”, SOMFA creates a
QSAR model based on molecular shape and electrostatic potentials (ESP)<sup><span
style='mso-field-code:" NOTEREF _Ref188303701 \\h &#1; \\* MERGEFORMAT "'>46<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003300300033003700300031000000</w:data>
</xml><![endif]--></span></sup>. 3D-QSAR methods that do not rely on a grid or
alignment are Autocorrelation of Molecular Surface Properties (AMSP)<a
style='mso-endnote-id:edn47' href="data_classification_qsar.htm#_edn47" name="_ednref47" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[47]</span></span><![endif]></span></span></a>,
Molecular Representation of Structures based on Electron diffraction (3D-MoRSE)<a
style='mso-endnote-id:edn48' href="data_classification_qsar.htm#_edn48" name="_ednref48" title=""><span
class=MsoEndnoteReference><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[48]</span></span><![endif]></span></span></a>, and
Comparative Molecular Moment Analysis (CoMMA)<a style='mso-endnote-id:edn49'
href="data_classification_qsar.htm#_edn49" name="_ednref49" title=""><span class=MsoEndnoteReference><span
style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA;
mso-bidi-font-style:italic'>[49]</span></span><![endif]></span></span></a>. The
methods of AMSP and 3D-MoRSE map the physical properties of the ligands to a
van der Waals surface and individual atoms, respectively. The physical
properties are transformed into a vector of equal length for all the ligands in
the study, eliminating the problem of molecular alignment. CoMMA removes the
alignment issue by calculating molecular moments and uses them as descriptors
to construct a QSAR model. The qualitative method of Probabilistic Receptor Potentials
(PRP)<sup><span style='mso-field-code:" NOTEREF _Ref188304414 \\h &#1; \\* MERGEFORMAT "'>41<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F005200650066003100380038003300300034003400310034000000</w:data>
</xml><![endif]--></span></sup> is able to predict the type of receptor atoms most
likely to interact with the ligand, thus providing an approximation of the type
of atoms in the active site. The technique of 3D-QSAR is built on methods that
take into consideration the 3D structure of the molecules and their inherent
physical properties (with respect to how the ligand will interact with the
receptor) to construct a QSAR model. All of these methods are
receptor-independent (RI); the models are created without any firm knowledge of
the composition or 3D structure of the receptor.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify'><b style='mso-bidi-font-weight:normal'><span style='font-size:12.0pt;
line-height:115%;mso-fareast-font-family:"Times New Roman";color:red;
mso-bidi-font-style:italic'>TASK 3.</span></b><b style='mso-bidi-font-weight:
normal'><span style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:
"Times New Roman";mso-bidi-font-style:italic'> </span></b><span
style='font-size:12.0pt;line-height:115%;mso-fareast-font-family:"Times New Roman";
mso-bidi-font-style:italic'>A comprehensive review on 3D-QSARs in lead design
has been recently written by Leitao <i>et al.</i> Please read it! (download
PDF3).<o:p></o:p></span></p>

</div>

<div style='mso-element:endnote-list'><![if !supportEndnotes]><br clear=all>

<hr align=left size=1 width="33%">

<![endif]>

<div style='mso-element:endnote' id=edn1>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn1' href="data_classification_qsar.htm#_ednref1"
name="_edn1" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[1]</span></span><![endif]></span></span></a>
Wold, S.; Sjöström, M.; Eriksson, L. <i style='mso-bidi-font-style:normal'>The
Encyclopedia of Comptational Chemistry</i>, Schleyer, P.v.R.; Allinger, N.L.;
Clark, T.; Gasteiger, J.; Kollman, P.A., Schaefer, H.F., III; Schreiner, P.R.
(Eds.), Wiley and Sons, Chichester, UK, 1998, pgs. 2006-2021.</p>

</div>

<div style='mso-element:endnote' id=edn2>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn2' href="data_classification_qsar.htm#_ednref2"
name="_edn2" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[2]</span></span><![endif]></span></span></a>
Siedlecki, W.; Siedlecka, K.; Sklansky, J. <i style='mso-bidi-font-style:normal'>Pattern
Recognition</i> <b style='mso-bidi-font-weight:normal'>1988</b>, <i
style='mso-bidi-font-style:normal'>21</i>, 411-429.</p>

</div>

<div style='mso-element:endnote' id=edn3>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn3' href="data_classification_qsar.htm#_ednref3"
name="_edn3" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[3]</span></span><![endif]></span></span></a>
Zupan, J.; Gasteiger, J. <i style='mso-bidi-font-style:normal'>Neural Networks
in Chemistry and Drug Design</i>, 2<sup>nd</sup> Edition, Wiley-VCH, Weinheim,
1999.</p>

</div>

<div style='mso-element:endnote' id=edn4>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn4' href="data_classification_qsar.htm#_ednref4"
name="_edn4" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[4]</span></span><![endif]></span></span></a>
Kohonen, T. <i style='mso-bidi-font-style:normal'>Self-Organizing Maps</i>,
Springer, Berlin, 1995.</p>

</div>

<div style='mso-element:endnote' id=edn5>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn5' href="data_classification_qsar.htm#_ednref5"
name="_edn5" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[5]</span></span><![endif]></span></span></a> Olah,
M.; Bologa, C.; Oprea, T.I. <i style='mso-bidi-font-style:normal'>J.
Comp.-Aided Mol. Design</i> <b style='mso-bidi-font-weight:normal'>2004</b>, <i
style='mso-bidi-font-style:normal'>18</i>, 437-449.</p>

</div>

<div style='mso-element:endnote' id=edn6>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn6' href="data_classification_qsar.htm#_ednref6"
name="_edn6" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[6]</span></span><![endif]></span></span></a>
Werther, W.; Demuth, W.; Krueger, F.R.; Kissel, J.; Schmid, E.R.; Varmuza, K. <i
style='mso-bidi-font-style:normal'>J. Chemometrics</i> <b style='mso-bidi-font-weight:
normal'>2002</b>, <i style='mso-bidi-font-style:normal'>16</i>, 99-110.</p>

</div>

<div style='mso-element:endnote' id=edn7>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn7' href="data_classification_qsar.htm#_ednref7"
name="_edn7" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[7]</span></span><![endif]></span></span></a>
Crawford, L.R.; Morrison, J.D. <i style='mso-bidi-font-style:normal'>Anal.
Chem.</i> <b style='mso-bidi-font-weight:normal'>1968</b>, <i style='mso-bidi-font-style:
normal'>40</i>, 1469-1474.</p>

</div>

<div style='mso-element:endnote' id=edn8>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn8' href="data_classification_qsar.htm#_ednref8"
name="_edn8" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[8]</span></span><![endif]></span></span></a>
Jurs, P.C.; Kowalski, B.R.; Isenhour, T.L. <i style='mso-bidi-font-style:normal'>Anal.
Chem.</i> <b style='mso-bidi-font-weight:normal'>1969</b>, <i style='mso-bidi-font-style:
normal'>41</i>, 21-27.</p>

</div>

<div style='mso-element:endnote' id=edn9>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn9' href="data_classification_qsar.htm#_ednref9"
name="_edn9" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[9]</span></span><![endif]></span></span></a>
Cristianini, N.; Shawe-Taylor, J. <i style='mso-bidi-font-style:normal'>An
Introduction to Support Vector Machines and other Kernel-Based Learning Methods</i>,
Cambridge University Press, UK, 2000.</p>

</div>

<div style='mso-element:endnote' id=edn10>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn10' href="data_classification_qsar.htm#_ednref10"
name="_edn10" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[10]</span></span><![endif]></span></span></a>
Brereton, R.G. <i style='mso-bidi-font-style:normal'>Chemometrics. Applications
of Mathematics and Statistics to Laboratory Systems</i>, Ellis Horwood, New
York, 1990.</p>

</div>

<div style='mso-element:endnote' id=edn11>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn11' href="data_classification_qsar.htm#_ednref11"
name="_edn11" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[11]</span></span><![endif]></span></span></a>
Wold, S. <i style='mso-bidi-font-style:normal'>Pattern Recognition</i> <b
style='mso-bidi-font-weight:normal'>1976</b>, <i style='mso-bidi-font-style:
normal'>8</i>, 127-139.</p>

</div>

<div style='mso-element:endnote' id=edn12>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn12' href="data_classification_qsar.htm#_ednref12"
name="_edn12" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[12]</span></span><![endif]></span></span></a>
Kvalheim, O.M.; Karstang, T.V. <i style='mso-bidi-font-style:normal'>Multivariate
Pattern Recognition in Chemometrics, Illustrated by Case Studies</i>, R.G.
Brereton (Eds.), Elsevier, Amsterdam, 1992, pgs. 209-248.</p>

</div>

<div style='mso-element:endnote' id=edn13>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn13' href="data_classification_qsar.htm#_ednref13"
name="_edn13" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[13]</span></span><![endif]></span></span></a>
Quinlan, J.R. <i style='mso-bidi-font-style:normal'>Machine Learning</i> <b
style='mso-bidi-font-weight:normal'>1986</b>, <i style='mso-bidi-font-style:
normal'>1</i>, 81-106.</p>

</div>

<div style='mso-element:endnote' id=edn14>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn14' href="data_classification_qsar.htm#_ednref14"
name="_edn14" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[14]</span></span><![endif]></span></span></a>
King, R.D.; Hirst, J.D.; Sternberg, M.J.E. <i style='mso-bidi-font-style:normal'>Appl.
Artif. Intell.</i> <b style='mso-bidi-font-weight:normal'>1994</b>, <i
style='mso-bidi-font-style:normal'>9</i>, 213-234.</p>

</div>

<div style='mso-element:endnote' id=edn15>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn15' href="data_classification_qsar.htm#_ednref15"
name="_edn15" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[15]</span></span><![endif]></span></span></a>
McGovern, S.L; Caselli, E.; Grigorieff, N.; Shoichet, B.K. <i style='mso-bidi-font-style:
normal'>J. Med. Chem.</i> <b style='mso-bidi-font-weight:normal'>2002</b>, <i
style='mso-bidi-font-style:normal'>45</i>, 1712-1722.</p>

</div>

<div style='mso-element:endnote' id=edn16>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn16' href="data_classification_qsar.htm#_ednref16"
name="_edn16" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[16]</span></span><![endif]></span></span></a>
McGovern, S.L.; Helfand, B.T.; Feng, B.; Shoichet, B.K. <i style='mso-bidi-font-style:
normal'>J. Med. Chem.</i> <b style='mso-bidi-font-weight:normal'>2003</b>, <i
style='mso-bidi-font-style:normal'>46</i>, 4265-4272.</p>

</div>

<div style='mso-element:endnote' id=edn17>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn17' href="data_classification_qsar.htm#_ednref17"
name="_edn17" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[17]</span></span><![endif]></span></span></a>
Neapolitan, R.E. <i style='mso-bidi-font-style:normal'>Probabilistic Reasoning
in Expert Systems. Theory and Algorithm</i>, Wiley, New York, 1990.</p>

</div>

<div style='mso-element:endnote' id=edn18>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn18' href="data_classification_qsar.htm#_ednref18"
name="_edn18" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[18]</span></span><![endif]></span></span></a>
Vapnik, V.N. <i style='mso-bidi-font-style:normal'>The Nature of Statistical
Learning Theory</i>, Springer, New York, 1995.</p>

</div>

<div style='mso-element:endnote' id=edn19>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn19' href="data_classification_qsar.htm#_ednref19"
name="_edn19" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[19]</span></span><![endif]></span></span></a>
Vapnik, V.N. <i style='mso-bidi-font-style:normal'>Statistical Learning Theory</i>,
Wiley, New York, 1998.</p>

</div>

<div style='mso-element:endnote' id=edn20>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn20' href="data_classification_qsar.htm#_ednref20"
name="_edn20" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[20]</span></span><![endif]></span></span></a> Demiriz,
A.; Bennett, K.P., Breneman, C.M. <i style='mso-bidi-font-style:normal'>Support
vector machine regression in chemometrics.</i> In <i style='mso-bidi-font-style:
normal'>Computing Science and Statistics</i>: Proceedings of Interface, 2001.</p>

</div>

<div style='mso-element:endnote' id=edn21>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn21' href="data_classification_qsar.htm#_ednref21"
name="_edn21" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[21]</span></span><![endif]></span></span></a>
Burbidge, R.; Trotter, M.; Buxton, B.; Holden, S. <i style='mso-bidi-font-style:
normal'>Drug design by machine learning: support vector machines for
pharmaceutical data analysis</i>, <i style='mso-bidi-font-style:normal'>Computers
and Chemistry</i>, <b style='mso-bidi-font-weight:normal'>2001</b>, <i
style='mso-bidi-font-style:normal'>26</i>, 4-15.</p>

</div>

<div style='mso-element:endnote' id=edn22>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn22' href="data_classification_qsar.htm#_ednref22"
name="_edn22" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[22]</span></span><![endif]></span></span></a> Trotter,
M.; Buxton, B.; Holden, S.B. <i style='mso-bidi-font-style:normal'>Support
Vector Machines in Combinatorial Chemistry, Measurement and Control,</i> <b
style='mso-bidi-font-weight:normal'>2001</b>, <i style='mso-bidi-font-style:
normal'>34</i>, 235-239.</p>

</div>

<div style='mso-element:endnote' id=edn23>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn23' href="data_classification_qsar.htm#_ednref23"
name="_edn23" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[23]</span></span><![endif]></span></span></a> Fara,
D.C. <i style='mso-bidi-font-style:normal'>QSPR Modeling of Complexation and
Distribution of Organic Compounds</i>, <i style='mso-bidi-font-style:normal'>Dissertationes
Chimicae Universitatis Tartuensis</i>, Tartu University Press, Tartu, 2004.<i
style='mso-bidi-font-style:normal'><o:p></o:p></i></p>

</div>

<div style='mso-element:endnote' id=edn24>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify;line-height:normal'><a style='mso-endnote-id:edn24' href="data_classification_qsar.htm#_ednref24"
name="_edn24" title=""><span class=MsoEndnoteReference><span style='font-size:
10.0pt'><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:10.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[24]</span></span><![endif]></span></span></span></a><span
style='font-size:10.0pt'> Lowe, J. P. <i style='mso-bidi-font-style:normal'>Quantum
Chemistry</i>; 2<sup>nd</sup> ed. Academic Press, San Diego, California, 1993.<o:p></o:p></span></p>

</div>

<div style='mso-element:endnote' id=edn25>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn25' href="data_classification_qsar.htm#_ednref25"
name="_edn25" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[25]</span></span><![endif]></span></span></a> Burkert,
U.; Allinger, N. L. (Eds.), <i>Molecular mechanics</i>, Am. Chem. Soc.
Monograph, Washington D.C., 1982.</p>

</div>

<div style='mso-element:endnote' id=edn26>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn26' href="data_classification_qsar.htm#_ednref26"
name="_edn26" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[26]</span></span><![endif]></span></span></a> Xu,
L.; Zhang, W.-J. <i style='mso-bidi-font-style:normal'>Anal. Chim. Acta</i> <b
style='mso-bidi-font-weight:normal'>2001</b>, <i style='mso-bidi-font-style:
normal'>446</i>, 477-483.</p>

</div>

<div style='mso-element:endnote' id=edn27>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn27' href="data_classification_qsar.htm#_ednref27"
name="_edn27" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[27]</span></span><![endif]></span></span></a> Deaven,
D. M.; Ho, K. M. <i style='mso-bidi-font-style:normal'>Phys. Rev. Lett.</i> <b
style='mso-bidi-font-weight:normal'>1995</b>, <i style='mso-bidi-font-style:
normal'>75</i>, 288.</p>

</div>

<div style='mso-element:endnote' id=edn28>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn28' href="data_classification_qsar.htm#_ednref28"
name="_edn28" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[28]</span></span><![endif]></span></span></a> Katritzky,
A. R.; Lobanov, V. S.; Karelson, M.; Murugan, R.; Grendze, M. P.; Toomey, J. E.
<i>Rev. Roum. Chim.</i> <b>1996</b>, <i>41</i>, 851-867.</p>

</div>

<div style='mso-element:endnote' id=edn29>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn29' href="data_classification_qsar.htm#_ednref29"
name="_edn29" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[29]</span></span><![endif]></span></span></a> Gabrielsson,
J.; Lindberg, N.-O.; Lundstedt, T. <i style='mso-bidi-font-style:normal'>J.
Chemom.</i> <b style='mso-bidi-font-weight:normal'>2002</b>, <i
style='mso-bidi-font-style:normal'>16</i>, 141-160.</p>

</div>

<div style='mso-element:endnote' id=edn30>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn30' href="data_classification_qsar.htm#_ednref30"
name="_edn30" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[30]</span></span><![endif]></span></span></a> Willett,
P. <i style='mso-bidi-font-style:normal'>Similarity and Clustering in Chemical
Information Systems</i>; John Wiley: New York, 1987.</p>

</div>

<div style='mso-element:endnote' id=edn31>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn31' href="data_classification_qsar.htm#_ednref31"
name="_edn31" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[31]</span></span><![endif]></span></span></a> Jackson,
J. E. <i>A users guide to principal components</i>; John Wiley &amp; Sons: New
York, 1991.</p>

</div>

<div style='mso-element:endnote' id=edn32>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn32' href="data_classification_qsar.htm#_ednref32"
name="_edn32" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[32]</span></span><![endif]></span></span></a> Schneider,
G.; Wrede, P. <i style='mso-bidi-font-style:normal'>Progr. Biophys. Mol. Biol.</i>
<b style='mso-bidi-font-weight:normal'>1998</b>, <i style='mso-bidi-font-style:
normal'>70</i>, 175-222.</p>

</div>

<div style='mso-element:endnote' id=edn33>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn33' href="data_classification_qsar.htm#_ednref33"
name="_edn33" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[33]</span></span><![endif]></span></span></a> Wold,
S. <i>Technometrics</i> <b>1978</b>, <i>20</i>, 397-405.</p>

</div>

<div style='mso-element:endnote' id=edn34>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn34' href="data_classification_qsar.htm#_ednref34"
name="_edn34" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[34]</span></span><![endif]></span></span></a> <i>SIMCA-P
8.0</i>; product of Umetrics AB, Umeå, Sweden, 2000; <a
href="http://www.umetrics.com/">http://www.umetrics.com/</a> </p>

</div>

<div style='mso-element:endnote' id=edn35>

<p class=MsoNormal style='margin-bottom:0in;margin-bottom:.0001pt;text-align:
justify;line-height:normal'><a style='mso-endnote-id:edn35' href="data_classification_qsar.htm#_ednref35"
name="_edn35" title=""><span class=MsoEndnoteReference><span style='font-size:
10.0pt'><span style='mso-special-character:footnote'><![if !supportFootnotes]><span
class=MsoEndnoteReference><span style='font-size:10.0pt;line-height:115%;
font-family:"Times New Roman","serif";mso-fareast-font-family:Calibri;
mso-ansi-language:EN-US;mso-fareast-language:EN-US;mso-bidi-language:AR-SA'>[35]</span></span><![endif]></span></span></span></a><span
style='font-size:10.0pt'> Katritzky, A. R.; Fara, D.; Tatham, D.; Petrukhin,
R.; Maran, U.; Lomaka, A.; Karelson, M. <i>Curr. Top. Med. Chem.</i> <b>2002</b>,
<i>2(12)</i>, 1333-1356.<o:p></o:p></span></p>

<p class=MsoEndnoteText><o:p>&nbsp;</o:p></p>

</div>

<div style='mso-element:endnote' id=edn36>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn36' href="data_classification_qsar.htm#_ednref36"
name="_edn36" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[36]</span></span><![endif]></span></span></a> Katritzky,
A. R.; Wang, Y.; Sild, S.; Tamm, T.; Karelson, M. <i>J. Chem. Inf. Comput. Sci.</i>
1998, <i>38</i>, 720.</p>

</div>

<div style='mso-element:endnote' id=edn37>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn37' href="data_classification_qsar.htm#_ednref37"
name="_edn37" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[37]</span></span><![endif]></span></span></a> Maran,
U.; Karelson, M.; Katritzky, A. R. <i>Quant. Struct. Act. Relat.</i> 1999, <i>18</i>,
3.</p>

</div>

<div style='mso-element:endnote' id=edn38>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn38' href="data_classification_qsar.htm#_ednref38"
name="_edn38" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[38]</span></span><![endif]></span></span></a> Allen,
D. M. <i>Technometrics</i> <b>1974</b>, <i>16</i>, 125-127.</p>

</div>

<div style='mso-element:endnote' id=edn39>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn39' href="data_classification_qsar.htm#_ednref39"
name="_edn39" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[39]</span></span><![endif]></span></span></a> Xu,
Q.-S.; Liang, Y.-Z. <i>Chemom. Intell. Lab. Syst..</i> <b>2001</b>, <i>56</i>,
1-11.</p>

</div>

<div style='mso-element:endnote' id=edn40>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn40' href="data_classification_qsar.htm#_ednref40"
name="_edn40" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[40]</span></span><![endif]></span></span></a>
Hopfinger, A.J. <i style='mso-bidi-font-style:normal'>J. Am. Chem. Soc.</i> <b
style='mso-bidi-font-weight:normal'>1980</b>, <i style='mso-bidi-font-style:
normal'>102</i>, 7196-7206.</p>

</div>

<div style='mso-element:endnote' id=edn41>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn41' href="data_classification_qsar.htm#_ednref41"
name="_edn41" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[41]</span></span><![endif]></span></span></a>
Labute, P. <i style='mso-bidi-font-style:normal'>J. Chem. Comput. Group</i> <b
style='mso-bidi-font-weight:normal'>2001</b>; <a
href="http://www.chemcomp.com/feature/cstat.htm">http://www.chemcomp.com/feature/cstat.htm</a>
</p>

</div>

<div style='mso-element:endnote' id=edn42>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn42' href="data_classification_qsar.htm#_ednref42"
name="_edn42" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[42]</span></span><![endif]></span></span></a>
Labute, P. <i style='mso-bidi-font-style:normal'>J. Chem. Comput. Group</i> <b
style='mso-bidi-font-weight:normal'>2001</b>; <a
href="http://www.chemcomp.com/feature/htsbqsar.htm">http://www.chemcomp.com/feature/htsbqsar.htm</a>
</p>

</div>

<div style='mso-element:endnote' id=edn43>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn43' href="data_classification_qsar.htm#_ednref43"
name="_edn43" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[43]</span></span><![endif]></span></span></a>
Hopfinger, A.J.; Reaka, A.; Venkatarangan, P.; Duca, J.S.; Wang, S. <i
style='mso-bidi-font-style:normal'>J. Chem. Inf. Comput. Sci.</i> <b
style='mso-bidi-font-weight:normal'>1999</b>, <i style='mso-bidi-font-style:
normal'>39</i>, 1151-1160.</p>

</div>

<div style='mso-element:endnote' id=edn44>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn44' href="data_classification_qsar.htm#_ednref44"
name="_edn44" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[44]</span></span><![endif]></span></span></a>
Cramer, R.D., III; Patterson, D.E.; Bunce, J.D. <i style='mso-bidi-font-style:
normal'>J. Am. Chem. Soc.</i> <b style='mso-bidi-font-weight:normal'>1998</b>, <i
style='mso-bidi-font-style:normal'>110</i>, 5959-5967.</p>

</div>

<div style='mso-element:endnote' id=edn45>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn45' href="data_classification_qsar.htm#_ednref45"
name="_edn45" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[45]</span></span><![endif]></span></span></a>
Klebe, G.; Abraham, U.; Mietzner, T. <i style='mso-bidi-font-style:normal'>J.
Med. Chem.</i> <b style='mso-bidi-font-weight:normal'>1994</b>, <i
style='mso-bidi-font-style:normal'>37</i>, 4130-4146.</p>

</div>

<div style='mso-element:endnote' id=edn46>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn46' href="data_classification_qsar.htm#_ednref46"
name="_edn46" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[46]</span></span><![endif]></span></span></a> Robinson,
D.D.; Winn, P.J.; Lyne, P.D.; Richards, W.G. <i style='mso-bidi-font-style:
normal'>J. Med. Chem.</i> <b style='mso-bidi-font-weight:normal'>1999</b>, <i
style='mso-bidi-font-style:normal'>42</i>, 573-583. SOMFA is available at http://bellatrix.pcl.ox.ac.uk/downloads/</p>

</div>

<div style='mso-element:endnote' id=edn47>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn47' href="data_classification_qsar.htm#_ednref47"
name="_edn47" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[47]</span></span><![endif]></span></span></a>
Wagener, M.; Sadowski, J.; Gasteiger, J. <i style='mso-bidi-font-style:normal'>J.
Am. Chem. Soc.</i> <b style='mso-bidi-font-weight:normal'>1995</b>, <i
style='mso-bidi-font-style:normal'>117</i>, 7769-7775.</p>

</div>

<div style='mso-element:endnote' id=edn48>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn48' href="data_classification_qsar.htm#_ednref48"
name="_edn48" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[48]</span></span><![endif]></span></span></a>
Schuur, J.H.; Seizer, P.; Gasteiger, J. <i style='mso-bidi-font-style:normal'>J.
Chem. Inf. Comput. Sci.</i> <b style='mso-bidi-font-weight:normal'>1996</b>, <i
style='mso-bidi-font-style:normal'>36</i>, 334-344.</p>

</div>

<div style='mso-element:endnote' id=edn49>

<p class=MsoEndnoteText><a style='mso-endnote-id:edn49' href="data_classification_qsar.htm#_ednref49"
name="_edn49" title=""><span class=MsoEndnoteReference><span style='mso-special-character:
footnote'><![if !supportFootnotes]><span class=MsoEndnoteReference><span
style='font-size:10.0pt;line-height:115%;font-family:"Times New Roman","serif";
mso-fareast-font-family:Calibri;mso-ansi-language:EN-US;mso-fareast-language:
EN-US;mso-bidi-language:AR-SA'>[49]</span></span><![endif]></span></span></a>
Silverman, B.D.; Platt, D.E. <i style='mso-bidi-font-style:normal'>J. Med.
Chem.</i> <b style='mso-bidi-font-weight:normal'>1996</b>, <i style='mso-bidi-font-style:
normal'>39</i>, 2129-2140.</p>

</div>

</div>

</body>

</html>
